---
layout: post
title: 2019/02/08 22 Updates (21 - 22)
---
#### [21. CERENKOV2: improved detection of functional noncoding SNPs using data-space geometric features.](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-019-2637-4){: .btn}
**Authors:** *Yao Yao, Liu Zheng, Wei Qi, Ramsey Stephen A*

**Journal:** *BMC bioinformatics*

*abstract:* We previously reported on CERENKOV, an approach for identifying regulatory single nucleotide polymorphisms (rSNPs) that is based on 246 annotation features. CERENKOV uses the xgboost classifier and is designed to be used to find causal noncoding SNPs in loci identified by genome-wide association studies (GWAS). We reported that CERENKOV has state-of-the-art performance (by two traditional measures and a novel GWAS-oriented measure, AVGRANK) in a comparison to nine other tools for identifying functional noncoding SNPs, using a comprehensive reference SNP set (OSU17, 15,331 SNPs). Given that SNPs are grouped within loci in the reference SNP set and given the importance of the data-space manifold geometry for machine-learning model selection, we hypothesized that within-locus inter-SNP distances would have class-based distributional biases that could be exploited to improve rSNP recognition accuracy. We thus defined an intralocus SNP "radius" as the average data-space distance from a SNP to the other intralocus neighbors, and explored radius likelihoods for five distance measures.

**Keywords:** **'Data space', 'GWAS', 'Machine learning', 'SNP', 'noncoding', 'rSNP'**

**Publication Date:** *2019-Feb-06*

#### [22. DeepPVP: phenotype-based prioritization of causative variants using deep learning.](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-019-2633-8){: .btn}
**Authors:** *Boudellioua Imane, Kulmanov Maxat, Schofield Paul N, Gkoutos Georgios V, Hoehndorf Robert*

**Journal:** *BMC bioinformatics*

*abstract:* Prioritization of variants in personal genomic data is a major challenge. Recently, computational methods that rely on comparing phenotype similarity have shown to be useful to identify causative variants. In these methods, pathogenicity prediction is combined with a semantic similarity measure to prioritize not only variants that are likely to be dysfunctional but those that are likely involved in the pathogenesis of a patient's phenotype.

**Keywords:** **'Machine learning', 'Ontology', 'Phenotype', 'Variant prioritization'**

**Publication Date:** *2019-Feb-06*

