---
layout: post
title: 2019/01/25 28 Updates (11 - 20)
---
#### [11. Predicting survival from colorectal cancer histology slides using deep learning: A retrospective multicenter study.](http://dx.plos.org/10.1371/journal.pmed.1002730){: .btn}
**Authors:** *Kather Jakob Nikolas, Krisam Johannes, Charoentong Pornpimol, Luedde Tom, Herpel Esther, Weis Cleo-Aron, Gaiser Timo, Marx Alexander, Valous Nektarios A, Ferber Dyke, Jansen Lina, Reyes-Aldasoro Constantino Carlos, Zörnig Inka, Jäger Dirk, Brenner Hermann, Chang-Claude Jenny, Hoffmeister Michael, Halama Niels*

**Journal:** *PLoS medicine*

*abstract:* For virtually every patient with colorectal cancer (CRC), hematoxylin-eosin (HE)-stained tissue slides are available. These images contain quantitative information, which is not routinely used to objectively extract prognostic biomarkers. In the present study, we investigated whether deep convolutional neural networks (CNNs) can extract prognosticators directly from these widely available images.



**Publication Date:** *2019-Jan*

#### [12. Deep Learning: Current and Emerging Applications in Medicine and Technology.](https://dx.doi.org/10.1109/JBHI.2019.2894713){: .btn}
**Authors:** *Akay Altug, Hess Henry*

**Journal:** *IEEE journal of biomedical and health informatics*

*abstract:* Machine learning is enabling researchers to analyze and understand increasingly complex physical and biological phenomena in traditional fields such as biology, medicine, and engineering and emerging fields like synthetic biology, automated chemical synthesis, and bio-manufacturing. These fields require new paradigms towards understanding increasingly complex data and converting such data into medical products and services for patients. The move towards deep learning and complex modeling is an attempt to bridge the gap between acquiring massive quantities of complex data, and converting such data into practical insights. Here, we provide an overview of the field of machine learning, its current applications and needs in traditional and emerging fields, and discuss an illustrative attempt at using deep learning to understand swarm behavior of molecular shuttles.



**Publication Date:** *2019-Jan-23*

#### [13. Improving Prediction Performance Using Hierarchical Analysis of Real-Time Data: A Sepsis Case Study.](https://dx.doi.org/10.1109/JBHI.2019.2894570){: .btn}
**Authors:** *van Wyk Franco, Khojandi Anahita, Kamaleswaran Rishikesan*

**Journal:** *IEEE journal of biomedical and health informatics*

*abstract:* This paper presents a novel method for hierarchical analysis of machine learning algorithms to improve predictions of at risk patients, thus further enabling prompt therapy. Specifically, we develop a multi-layer machine learning approach to analyze continuous, high-frequency data. We illustrate the capabilities of this approach for early identification of patients at risk of sepsis, a potentially life-threatening complication of an infection, using high-frequency (minute-by-minute) physiological data collected from bedside monitors. In our analysis of a cohort of 586 patients, the model obtained from analyzing the output of a previously developed sepsis prediction model resulted in improved outcomes. Specifically, the original model failed to predict 11.76 ± 4.26% of sepsis patients earlier than Systemic Inflammatory Response Syndrome (SIRS) criteria, commonly used to identify patients at risk for rapid physiological deterioration resulting from sepsis. In contrast, the multi-layer model only failed to predict 3.21 ± 3.11% of sepsis patients earlier than SIRS. In addition, sepsis patients were predicted on average 204.87 ± 7.90 minutes earlier than SIRS criteria using the multi-layer model, which can potentially help reduce mortality and morbidity if implemented in the ICU.



**Publication Date:** *2019-Jan-23*

#### [14. Graph-based Hub Gene Selection Technique using Protein Interaction Information: Application to Sample Classification.](https://dx.doi.org/10.1109/JBHI.2019.2894374){: .btn}
**Authors:** *Dutta Pratik, Saha Sriparna, Gulati Saurabh*

**Journal:** *IEEE journal of biomedical and health informatics*

*abstract:* Classification of samples of gene expression profile plays a significant role in prediction and diagnosis of diseases. In the task of sample classification, a robust feature selection algorithm is very much essential to identify the important genes from the high dimensional gene expression data. This paper explores the information of protein-protein interaction (PPI) with a graph mining technique for finding a proper subset of features (genes), which further takes part in sample classification. Here, our contribution for feature selection is three-fold: first, all the genes are grouped into different clusters based on the integrated information of the gene expression values and their protein interactions using a multi-objective optimization (MOO) based clustering approach. Secondly, the confidence scores of the protein-protein interactions are incorporated in a popular graph mining algorithm namely Goldberg algorithm to find out the relevant features. These features are the topologically and functionally significant genes, named as hub genes. Finally, these hub genes are identified varying the degrees of the nodes, and those are utilized for the sample classification task. Different machine learning classifiers are exploited for this purpose, and the classification performance is measured with respect to various performance metrics namely accuracy, sensitivity, specificity, precision, F-measure and Mathews coefficient correlation (MCC). Comparative analysis with respect to two baselines and several existing approaches proves the efficiency of the proposed approach. Further, the robustness of the identified hub-gene modules is endorsed using some strong biological significance analysis.



**Publication Date:** *2019-Jan-21*

#### [15. A Data-driven Approach to Predict and Classify Epileptic Seizures from Brain-wide Calcium Imaging Video Data.](https://dx.doi.org/10.1109/TCBB.2019.2895077){: .btn}
**Authors:** *Zheng Jingyi, Fushing Hsieh, Ge Linqiang*

**Journal:** *IEEE/ACM transactions on computational biology and bioinformatics*

*abstract:* The prediction of epileptic seizures has been an essential problem of epilepsy study. The calcium imaging video data images the whole brain-wide neurons activities with electrical discharge recorded by calcium fluorescence intensity (CFI). In this paper, using the zebrafish's brain-wide calcium image video data, we propose a data-driven approach to effectively detect the systemic change-point, and further predict the epileptic seizures. Our approach includes two phases: offline training and online testing. Specifically, during offline training, we extract features and confirm the existence of systemic change-point, then estimate the ratio of unchanged system duration to interictal period duration. For online testing, we implement a statistical model to estimate the change-point, and then predict the onset of epileptic seizure. The testing results show that our proposed approach could effectively predict the time range of future epileptic seizure. Furthermore, we explore the macroscopic patterns of epileptic and control cases, and extract features based on the pattern difference, then implement and compare the classification performance from four machine learning models. Based on the data structure, we also propose a new method to discretize related features, and combine with hierarchical clustering to better visualize and explain the pattern difference between epileptic and control cases.



**Publication Date:** *2019-Jan-24*

#### [16. Mutual Component Convolutional Neural Networks for Heterogeneous Face Recognition.](https://dx.doi.org/10.1109/TIP.2019.2894272){: .btn}
**Authors:** *Deng Zhongying, Peng Xiaojiang, Li Zhifeng, Qiao Yu*

**Journal:** *IEEE transactions on image processing : a publication of the IEEE Signal Processing Society*

*abstract:* HHeterogeneous face recognition (HFR) aims to identify a person from different facial modalities such as visible and near-infrared images. The main challenges of HFR lie in the large modality discrepancy and insufficient training samples. In this paper, we propose the Mutual Component Convolutional Neural Network (MC-CNN), a modal-invariant deep learning framework, to tackle these two issues simultaneously. Our MCCNN incorporates a generative module, i.e. the Mutual Component Analysis (MCA) [1], into modern deep convolutional neural networks by viewing MCA as a special fully-connected (FC) layer. Based on deep features, this FC layer is designed to extract modal-independent hidden factors, and is updated according to maximum likelihood analytic formulation instead of back propagation which prevents over-fitting from limited data naturally. In addition, we develop an MCA loss to update the network for modal-invariant feature learning. Extensive experiments show that our MC-CNN outperforms several finetuned baseline models significantly. Our methods achieve the state-of-the-art performance on CASIA NIR-VIS 2.0, CUHK NIR-VIS and IIIT-D Sketch dataset.



**Publication Date:** *2019-Jan-23*

#### [17. Lung and Pancreatic Tumor Characterization in the Deep Learning Era: Novel Supervised and Unsupervised Learning Approaches.](https://dx.doi.org/10.1109/TMI.2019.2894349){: .btn}
**Authors:** *Hussein Sarfaraz, Kandel Pujan, Bolan Candice W, Wallace Michael B, Bagci Ulas*

**Journal:** *IEEE transactions on medical imaging*

*abstract:* Risk stratification (characterization) of tumors from radiology images can be more accurate and faster with computeraided diagnosis (CAD) tools. Tumor characterization through such tools can also enable non-invasive cancer staging, prognosis, and foster personalized treatment planning as a part of precision medicine. In this study, we propose both supervised and unsupervised machine learning strategies to improve tumor characterization. Our first approach is based on supervised learning for which we demonstrate significant gains with deep learning algorithms, particularly by utilizing a 3D Convolutional Neural Network and Transfer Learning. Motivated by the radiologists' interpretations of the scans, we then show how to incorporate task dependent feature representations into a CAD system via a graph-regularized sparse Multi-Task Learning (MTL) framework. In the second approach, we explore an unsupervised learning algorithm to address the limited availability of labeled training data, a common problem in medical imaging applications. Inspired by learning from label proportion (LLP) approaches in computer vision, we propose to use proportion-SVM for characterizing tumors. We also seek the answer to the fundamental question about the goodness of "deep features" for unsupervised tumor classification. We evaluate our proposed supervised and unsupervised learning algorithms on two different tumor diagnosis challenges: lung and pancreas with 1018 CT and 171 MRI scans, respectively, and obtain the state-of-the-art sensitivity and specificity results in both problems.



**Publication Date:** *2019-Jan-23*

#### [18. Automatic 3D bi-ventricular segmentation of cardiac images by a shape-refined multi-task deep learning approach.](https://dx.doi.org/10.1109/TMI.2019.2894322){: .btn}
**Authors:** *Duan Jinming, Bello Ghalib, Schlemper Jo, Bai Wenjia, Dawes Timothy J W, Biffi Carlo, de Marvao Antonio, Doumou Georgia, O'Regan Declan P, Rueckert Daniel*

**Journal:** *IEEE transactions on medical imaging*

*abstract:* Deep learning approaches have achieved state-of-the-art performance in cardiac magnetic resonance (CMR) image segmentation. However, most approaches have focused on learning image intensity features for segmentation, whereas the incorporation of anatomical shape priors has received less attention. In this paper, we combine a multi-task deep learning approach with atlas propagation to develop a shape-refined bi-ventricular segmentation pipeline for short-axis CMR volumetric images. The pipeline first employs a fully convolutional network (FCN) that learns segmentation and landmark localisation tasks simultaneously. The architecture of the proposed FCN uses a 2.5D representation, thus combining the computational advantage of 2D FCNs networks and the capability of addressing 3D spatial consistency without compromising segmentation accuracy. Moreover, a refinement step is designed to explicitly impose shape prior knowledge and improve segmentation quality. This step is effective for overcoming image artefacts (e.g. due to different breath-hold positions and large slice thickness), which preclude the creation of anatomically meaningful 3D cardiac shapes. The pipeline is fully automated, due to network's ability to infer landmarks, which are then used downstream in the pipeline to initialise atlas propagation. We validate the pipeline on 1831 healthy subjects and 649 subjects with pulmonary hypertension. Extensive numerical experiments on the two datasets demonstrate that our proposed method is robust and capable of producing accurate, high-resolution and anatomically smooth bi-ventricular 3D models, despite the presence of artefacts in input CMR volumes.



**Publication Date:** *2019-Jan-23*

#### [19. Screening Glaucoma With Red-free Fundus Photography Using Deep Learning Classifier and Polar Transformation.](http://dx.doi.org/10.1097/IJG.0000000000001187){: .btn}
**Authors:** *Lee Jinho, Kim Youngwoo, Kim Jong Hyo, Park Ki Ho*

**Journal:** *Journal of glaucoma*

*abstract:* To develop novel software to determine whether there is a retinal nerve fiber layer (RNFL) defect in a given fundus image using deep learning classifier and, if there is, where it presents.



**Publication Date:** *2019-Jan-22*

#### [20. Sprint Assessment Using Machine Learning and a Wearable Accelerometer.](http://journals.humankinetics.com/doi/full/10.1123/jab.2018-0107?url_ver=Z39.88-2003&rfr_id=ori:rid:crossref.org&rfr_dat=cr_pub%3dpubmed){: .btn}
**Authors:** *Gurchiek Reed D, Rupasinghe Arachchige Don Hasthika S, Pelawa Watagoda Lasanthi C R, McGinnis Ryan S, van Werkhoven Herman, Needle Alan R, McBride Jeffrey M, Arnholt Alan T*

**Journal:** *Journal of applied biomechanics*

*abstract:* Field-based sprint performance assessments rely on metrics derived from a simple model of sprinting dynamics parameterized by 2 constants, v

**Keywords:** **'inertial sensor', 'sprint assessment', 'statistical learning', 'wearable sensor'**

**Publication Date:** *2019-Feb-05*

