---
layout: post
title: 2019/02/09 24 Updates (1 - 10)
---
#### [1. Non-invasive machine learning estimation of effort differentiates sleep-disordered breathing pathology.](https://doi.org/10.1088/1361-6579/ab0559){: .btn}
**Authors:** *Hanif Umaer, Schneider Logan Douglas, Trap Lotte, Leary Eileen B, Moore Iv Hyatt, Guilleminault Christian, Jennum Poul Jørgen, Bjarup Dissing Sørensen Helge, Mignot Emmanuel J M*

**Journal:** *Physiological measurement*

*abstract:* Obstructive sleep-disordered breathing (SDB) events, unlike central events, are associated with increased respiratory effort. Esophageal pressure (Pes) monitoring is the gold standard for measuring respiratory effort, but it is typically poorly tolerated because of its invasive nature. The objective was to investigate whether machine learning can be applied to routinely collected non-invasive, polysomnography (PSG) measures to accurately model peak negative Pes.&#13; Approach: 1119 patients from the Stanford Sleep Clinic with PSGs containing Pes served as the sample. The selected non-invasive PSG signals included nasal pressure, oral airflow, thoracoabdominal effort, and snoring. A long short-term memory (LSTM) neural network was implemented to achieve a context-based mapping between the non-invasive features and the Pes values. A hold-out dataset served as a prospective validation of the algorithm without needing to undertake a costly new study with the impractically invasive Pes. &#13; Main results: The median difference between the measured and predicted Pes was 0.61 cmH2O with an interquartile range (IQR) of 2.99 cmH2O and 5th and 95th percentiles of -5.85 cmH2O and 5.47 cmH2O, respectively. The model performed well when compared to actual esophageal pressure signal (ρmedian=0.581, p=0.01; IQR = 0.298; ρ5% = 0.106; ρ95% = 0.843).&#13; Significance: A significant difference in predicted Pes was shown between normal breathing and all obstructive SDB events; whereas, central apneas did not significantly differ from normal breathing. The developed system may be used as a tool for quantifying respiratory effort from the existing clinical practice of PSG without the need for Pes, improving characterization of SDB events as obstructive or not.&#13.

**Keywords:** **'Esophageal pressure', 'Machine learning', 'Sleep-disordered breathing'**

**Publication Date:** *2019-Feb-08*

#### [2. A deep learning framework for identifying essential proteins by integrating multiple types of biological information.](https://dx.doi.org/10.1109/TCBB.2019.2897679){: .btn}
**Authors:** *Zeng Min, Li Min, Fei Zhihui, Wu Fangxiang, Li Yaohang, Pan Yi, Wang Jianxin*

**Journal:** *IEEE/ACM transactions on computational biology and bioinformatics*

*abstract:* Computational methods including centrality and machine learning-based methods have been proposed to identify essential proteins for understanding the minimum requirements of the survival and evolution of a cell. In centrality methods, researchers are required to design a score function which is based on prior knowledge, yet is usually not sufficient to capture the complexity of biological information. In machine learning-based methods, some selected biological features cannot represent the complete properties of biological information as they lack a computational framework to automatically select features. To tackle these problems, we propose a deep learning framework to automatically learn biological features without prior knowledge. We use node2vec technique to automatically learn a richer representation of protein-protein interaction (PPI) network topologies than a score function. Bidirectional long short term memory cells are applied to capture non-local relationships in gene expression data. For subcellular localization information, we exploit a high dimensional indicator vector to characterize their feature. To evaluate the performance of our method, we tested it on PPI network of S. cerevisiae. Our experimental results demonstrate that the performance of our method is better than traditional centrality methods and is superior to existing machine learning-based methods. To explore which of the three types of biological information is the most vital element, we conduct an ablation study by removing each component in turn. Our results show that the PPI network embedding contributes most to the improvement. In addition, gene expression profiles and subcellular localization information are also helpful to improve the performance in identification of essential proteins.



**Publication Date:** *2019-Feb-05*

#### [3. EuroCity Persons: A Novel Benchmark for Person Detection in Traffic Scenes.](https://dx.doi.org/10.1109/TPAMI.2019.2897684){: .btn}
**Authors:** *Braun Markus, Krebs Sebastian, Flohr Fabian, Gavrila Dariu*

**Journal:** *IEEE transactions on pattern analysis and machine intelligence*

*abstract:* Big data has had a great share in the success of deep learning in computer vision. Recent works suggest that there is significant further potential to increase object detection performance by utilizing even bigger datasets. In this paper, we introduce the EuroCity Persons dataset, which provides a large number of highly diverse, accurate and detailed annotations of pedestrians, cyclists and other riders in urban traffic scenes. The images for this dataset were collected on-board a moving vehicle in 31 cities of 12 European countries. With over 238200 person instances manually labeled in over 47300 images, EuroCity Persons is nearly one order of magnitude larger than datasets used previously for person detection in traffic scenes. The dataset furthermore contains a large number of person orientation annotations (over 211200). We optimize four state-of-the-art deep learning approaches (Faster R-CNN, R-FCN, SSD and YOLOv3) to serve as baselines for the new object detection benchmark. We analyze the generalization capabilities of these detectors when trained with the new dataset. We furthermore study the effect of the training set size, the dataset diversity (day- vs. night-time, geographical region), the dataset detail (i.e. availability of object orientation information) and the annotation quality on the detector performance. Finally, we analyze error sources and discuss the road ahead.



**Publication Date:** *2019-Feb-05*

#### [4. Prediction of IDH1 Mutation Status in the Glioblastoma Using the Machine Learning Technique Based on the Quantitative Radiomic Data.](https://linkinghub.elsevier.com/retrieve/pii/S1878-8750(19)30257-8){: .btn}
**Authors:** *Lee Min Ho, Kim Junhyung, Kim Sung-Tae, Shin Hye-Mi, You Hye-Jin, Choi Jung Won, Seol Ho Jun, Nam Do-Hyun, Lee Jung-Il, Kong Doo-Sik*

**Journal:** *World neurosurgery*

*abstract:* Isocitrate dehydrogenase 1 (IDH1) mutation status is an independent favourable prognostic factor for glioblastoma (GBM) and is usually determined by sequencing or immunohistochemistry. An accurate prediction of IDH1 mutation status via noninvasive methods helps establish the appropriate treatment strategy. We aimed to predict IDH1 mutation status using quantitative radiomic data in patients with GBM.

**Keywords:** **'Glioblastoma · Isocitrate Dehydrogenase · Mutation', 'Machine Learning · Magnetic Resonance Imaging'**

**Publication Date:** *2019-Feb-05*

#### [5. Non-invasive neurophysiological measures of learning: A meta-analysis.](https://linkinghub.elsevier.com/retrieve/pii/S0149-7634(18)30420-2){: .btn}
**Authors:** *Tinga Angelica M, de Back Tycho T, Louwerse Max M*

**Journal:** *Neuroscience and biobehavioral reviews*

*abstract:* In a meta-analysis of 113 experiments we examined neurophysiological outcomes of learning, and the relationship between neurophysiological and behavioral outcomes of learning. Findings showed neurophysiology yielding large effect sizes, with the majority of studies examining electroencephalography and eye-related outcome measures. Effect sizes on neurophysiological outcomes were smaller than effect sizes on behavioral outcomes, however, neurophysiological outcomes were, but behavioral outcomes were not, influenced by several modulating factors. These factors included the sensory system in which learning took place, number of learning days, whether feedback on performance was provided, and age of participants. Controlling for these factors resulted in the effect size differences between behavior and neurophysiology to disappear. The findings of the current meta-analysis demonstrate that neurophysiology is an appropriate measure in assessing learning, particularly when taking into account factors that could have an influence on neurophysiology. We propose a first model to aid further studies that are needed to examine the exact interplay between learning, neurophysiology, behavior, individual differences, and task-related aspects.

**Keywords:** **'Behavior', 'Effect sizes', 'Electrocardiogram', 'Electrodermal activity', 'Electroencephalography', 'Eye tracking', 'Functional near-infrared spectroscopy', 'Learning', 'Meta-analysis', 'Neurophysiology', 'Respiration'**

**Publication Date:** *2019-Feb-05*

#### [6. Stacked Generalization with Applicability Domain Outperforms simple QSAR on in vitro Toxicological Data.](https://dx.doi.org/10.1021/acs.jcim.8b00553){: .btn}
**Authors:** *Grenet Ingrid, Merlo Kevin, Comet Jean Paul, Tertiaux Romain, Rouquié David, Dayan Frederic*

**Journal:** *Journal of chemical information and modeling*

*abstract:* The development of in silico tools able to predict bioactivity and toxicity of chemical substances is a powerful solution envisioned to asess toxicity as early as possible. To enable the development of such tools, the ToxCast program has generated and made publicly available in vitro bioactivity data for thousands of compounds. The goal of the present study is to characterize and explore the data from ToxCast in terms of Machine Learning capacity. For this, a large scale analysis on the entire database has been performed to build models to predict bioactivities measured in in vitro assays. Simple classical QSAR algorithms (ANN, SVM, LDA, Random Forest and Bayesian) were first applied on the data, and the results of these algorithms suggested that they do not seem to be well suited for datasets with a high proportion of inactive compounds. The study then showed for the first time that the use of an ensemble method named "Stacked generalization" could improve the model performance on this type of data. Indeed, for 61% of 483 models, the Stacked method led to models with higher performance. Moreover, the combination of this ensemble method with an applicability domain filter allows one to assess the reliability of the predictions for further compound prioritization. In particular we showed that for 50% of the models, the ROC score is better if we do not consider the compounds that are not within the applicability domain.



**Publication Date:** *2019-Feb-08*

#### [7. Tumour Parcellation and Quantification (TuPaQ): a tool for refining biomarker analysis through rapid and automated segmentation of tumour epithelium.](https://doi.org/10.1111/his.13838){: .btn}
**Authors:** *Abdelsamea Mohammed M, Grineviciute Ruta Barbora, Besusparis Justinas, Cham Sulayman, Pitiot Alain, Laurinavicius Arvydas, Ilyas Mohammad*

**Journal:** *Histopathology*

*abstract:* Immunohistochemistry (IHC) is an essential component of biomarker research in cancer. Automated biomarker quantification is hampered by the failure of computational algorithms to discriminate "negative" tumour cells from "negative" stromal cells. We sought to develop an algorithm for segmentation of tumour epithelium in colorectal cancer (CRC) irrespective of the biomarker expression in the cells.

**Keywords:** **'Biomarker analysis', 'Colorectal Cancer', 'Image Segmentation', 'Machine Learning', 'Parcellation'**

**Publication Date:** *2019-Feb-08*

#### [8. Estimating Retinal Sensitivity Using Optical Coherence Tomography With Deep-Learning Algorithms in Macular Telangiectasia Type 2.](https://jamanetwork.com/journals/jamanetworkopen/fullarticle/10.1001/jamanetworkopen.2018.8029){: .btn}
**Authors:** *Kihara Yuka, Heeren Tjebo F C, Lee Cecilia S, Wu Yue, Xiao Sa, Tzaridis Simone, Holz Frank G, Charbel Issa Peter, Egan Catherine A, Lee Aaron Y*

**Journal:** *JAMA network open*

*abstract:* As currently used, microperimetry is a burdensome clinical testing modality for testing retinal sensitivity requiring long testing times and trained technicians.



**Publication Date:** *2019-Feb-01*

#### [9. Detection of Bleeding Events in Electronic Health Record Notes Using Convolutional Neural Network Models Enhanced With Recurrent Neural Network Autoencoders: Deep Learning Approach.](http://medinform.jmir.org/2019/1/e10788/){: .btn}
**Authors:** *Li Rumeng, Hu Baotian, Liu Feifan, Liu Weisong, Cunningham Francesca, McManus David D, Yu Hong*

**Journal:** *JMIR medical informatics*

*abstract:* Bleeding events are common and critical and may cause significant morbidity and mortality. High incidences of bleeding events are associated with cardiovascular disease in patients on anticoagulant therapy. Prompt and accurate detection of bleeding events is essential to prevent serious consequences. As bleeding events are often described in clinical notes, automatic detection of bleeding events from electronic health record (EHR) notes may improve drug-safety surveillance and pharmacovigilance.

**Keywords:** **'BiLSTM', 'autoencoder', 'bleeding', 'convolutional neural networks', 'electronic health record'**

**Publication Date:** *2019-Feb-08*

#### [10. U-Net-based Deep-Learning Bladder Segmentation in CT Urography.](https://doi.org/10.1002/mp.13438){: .btn}
**Authors:** *Ma Xiangyuan, Hadjiiski Lubomir M, Wei Jun, Chan Heang-Ping, Cha Kenny H, Cohan Richard H, Caoili Elaine M, Samala Ravi, Zhou Chuan, Lu Yao*

**Journal:** *Medical physics*

*abstract:* To develop a U-Net based deep learning approach (U-DL) for bladder segmentation in computed tomography urography (CTU) as a part of a computer-assisted bladder cancer detection and treatment response assessment pipeline.

**Keywords:** **'Bladder', 'CT Urography', 'Computer-Aided Detection', 'Deep-Learning', 'Segmentation'**

**Publication Date:** *2019-Feb-08*

