---
layout: post
title: 2019/02/02 32 Updates (21 - 30)
---
## 21. A feasibility study for predicting optimal radiation therapy dose distributions of prostate cancer patients from patient anatomy using deep learning.


*Journal: Scientific reports* *Publication Date: 2019-Jan-31*

[http://dx.doi.org/10.1038/s41598-018-37741-x](http://dx.doi.org/10.1038/s41598-018-37741-x){: .btn}

*abstract:* With the advancement of treatment modalities in radiation therapy for cancer patients, outcomes have improved, but at the cost of increased treatment plan complexity and planning time. The accurate prediction of dose distributions would alleviate this issue by guiding clinical plan optimization to save time and maintain high quality plans. We have modified a convolutional deep network model, U-net (originally designed for segmentation purposes), for predicting dose from patient image contours of the planning target volume (PTV) and organs at risk (OAR). We show that, as an example, we are able to accurately predict the dose of intensity-modulated radiation therapy (IMRT) for prostate cancer patients, where the average Dice similarity coefficient is 0.91 when comparing the predicted vs. true isodose volumes between 0% and 100% of the prescription dose. The average value of the absolute differences in [max, mean] dose is found to be under 5% of the prescription dose, specifically for each structure is [1.80%, 1.03%](PTV), [1.94%, 4.22%](Bladder), [1.80%, 0.48%](Body), [3.87%, 1.79%](L Femoral Head), [5.07%, 2.55%](R Femoral Head), and [1.26%, 1.62%](Rectum) of the prescription dose. We thus managed to map a desired radiation dose distribution from a patient's PTV and OAR contours. As an additional advantage, relatively little data was used in the techniques and models described in this paper.

## 22. Discovering highly selective and diverse PPAR-delta agonists by ligand based machine learning and structural modeling.


*Journal: Scientific reports* *Publication Date: 2019-Jan-31*

[http://dx.doi.org/10.1038/s41598-019-38508-8](http://dx.doi.org/10.1038/s41598-019-38508-8){: .btn}

*abstract:* PPAR-δ agonists are known to enhance fatty acid metabolism, preserving glucose and physical endurance and are suggested as candidates for treating metabolic diseases. None have reached the clinic yet. Our Machine Learning algorithm called "Iterative Stochastic Elimination" (ISE) was applied to construct a ligand-based multi-filter ranking model to distinguish between confirmed PPAR-δ agonists and random molecules. Virtual screening of 1.56 million molecules by this model picked ~2500 top ranking molecules. Subsequent docking to PPAR-δ structures was mainly evaluated by geometric analysis of the docking poses rather than by energy criteria, leading to a set of 306 molecules that were sent for testing in vitro. Out of those, 13 molecules were found as potential PPAR-δ agonist leads with EC

## 23. Multi-Channel 3D Deep Feature Learning for Survival Time Prediction of Brain Tumor Patients Using Multi-Modal Neuroimages.


*Journal: Scientific reports* *Publication Date: 2019-Jan-31*

[http://dx.doi.org/10.1038/s41598-018-37387-9](http://dx.doi.org/10.1038/s41598-018-37387-9){: .btn}

*abstract:* High-grade gliomas are the most aggressive malignant brain tumors. Accurate pre-operative prognosis for this cohort can lead to better treatment planning. Conventional survival prediction based on clinical information is subjective and could be inaccurate. Recent radiomics studies have shown better prognosis by using carefully-engineered image features from magnetic resonance images (MRI). However, feature engineering is usually time consuming, laborious and subjective. Most importantly, the engineered features cannot effectively encode other predictive but implicit information provided by multi-modal neuroimages. We propose a two-stage learning-based method to predict the overall survival (OS) time of high-grade gliomas patient. At the first stage, we adopt deep learning, a recently dominant technique of artificial intelligence, to automatically extract implicit and high-level features from multi-modal, multi-channel preoperative MRI such that the features are competent of predicting survival time. Specifically, we utilize not only contrast-enhanced T1 MRI, but also diffusion tensor imaging (DTI) and resting-state functional MRI (rs-fMRI), for computing multiple metric maps (including various diffusivity metric maps derived from DTI, and also the frequency-specific brain fluctuation amplitude maps and local functional connectivity anisotropy-related metric maps derived from rs-fMRI) from 68 high-grade glioma patients with different survival time. We propose a multi-channel architecture of 3D convolutional neural networks (CNNs) for deep learning upon those metric maps, from which high-level predictive features are extracted for each individual patch of these maps. At the second stage, those deeply learned features along with the pivotal limited demographic and tumor-related features (such as age, tumor size and histological type) are fed into a support vector machine (SVM) to generate the final prediction result (i.e., long or short overall survival time). The experimental results demonstrate that this multi-model, multi-channel deep survival prediction framework achieves an accuracy of 90.66%, outperforming all the competing methods. This study indicates highly demanded effectiveness on prognosis of deep learning technique in neuro-oncological applications for better individualized treatment planning towards precision medicine.

## 24. Theories of Error Back-Propagation in the Brain.
**Keywords:** **'deep learning', 'neural networks', 'predictive coding', 'synaptic plasticity'**

*Journal: Trends in cognitive sciences* *Publication Date: 2019-Jan-28*

[https://linkinghub.elsevier.com/retrieve/pii/S1364-6613(19)30012-9](https://linkinghub.elsevier.com/retrieve/pii/S1364-6613(19)30012-9){: .btn}

*abstract:* This review article summarises recently proposed theories on how neural circuits in the brain could approximate the error back-propagation algorithm used by artificial neural networks. Computational models implementing these theories achieve learning as efficient as artificial neural networks, but they use simple synaptic plasticity rules based on activity of presynaptic and postsynaptic neurons. The models have similarities, such as including both feedforward and feedback connections, allowing information about error to propagate throughout the network. Furthermore, they incorporate experimental evidence on neural connectivity, responses, and plasticity. These models provide insights on how brain networks might be organised such that modification of synaptic weights on multiple levels of cortical hierarchy leads to improved performance on tasks.

## 25. How far have we come? Artificial intelligence for chest radiograph interpretation.


*Journal: Clinical radiology* *Publication Date: 2019-Jan-28*

[https://linkinghub.elsevier.com/retrieve/pii/S0009-9260(19)30019-4](https://linkinghub.elsevier.com/retrieve/pii/S0009-9260(19)30019-4){: .btn}

*abstract:* Due to recent advances in artificial intelligence, there is renewed interest in automating interpretation of imaging tests. Chest radiographs are particularly interesting due to many factors: relatively inexpensive equipment, importance to public health, commonly performed throughout the world, and deceptively complex taking years to master. This article presents a brief introduction to artificial intelligence, reviews the progress to date in chest radiograph interpretation, and provides a snapshot of the available datasets and algorithms available to chest radiograph researchers. Finally, the limitations of artificial intelligence with respect to interpretation of imaging studies are discussed.

## 26. Selecting precise reference normal tissue samples for cancer research using a deep learning approach.
**Keywords:** **'Autoencoder', 'Deep learning', 'Disease signatures', 'Drug repositioning'**

*Journal: BMC medical genomics* *Publication Date: 2019-Jan-31*

[https://bmcmedgenomics.biomedcentral.com/articles/10.1186/s12920-018-0463-6](https://bmcmedgenomics.biomedcentral.com/articles/10.1186/s12920-018-0463-6){: .btn}

*abstract:* Normal tissue samples are often employed as a control for understanding disease mechanisms, however, collecting matched normal tissues from patients is difficult in many instances. In cancer research, for example, the open cancer resources such as TCGA and TARGET do not provide matched tissue samples for every cancer or cancer subtype. The recent GTEx project has profiled samples from healthy individuals, providing an excellent resource for this field, yet the feasibility of using GTEx samples as the reference remains unanswered.

## 27. Predicting drug response of tumors from integrated genomic profiles by deep neural networks.
**Keywords:** **'Cancer cell line encyclopedia', 'Deep neural networks', 'Drug response prediction', 'Genomics of Drug Sensitivity in Cancer', 'Pharmacogenomics', 'The Cancer Genome Atlas'**

*Journal: BMC medical genomics* *Publication Date: 2019-Jan-31*

[https://bmcmedgenomics.biomedcentral.com/articles/10.1186/s12920-018-0460-9](https://bmcmedgenomics.biomedcentral.com/articles/10.1186/s12920-018-0460-9){: .btn}

*abstract:* The study of high-throughput genomic profiles from a pharmacogenomics viewpoint has provided unprecedented insights into the oncogenic features modulating drug response. A recent study screened for the response of a thousand human cancer cell lines to a wide collection of anti-cancer drugs and illuminated the link between cellular genotypes and vulnerability. However, due to essential differences between cell lines and tumors, to date the translation into predicting drug response in tumors remains challenging. Recently, advances in deep learning have revolutionized bioinformatics and introduced new techniques to the integration of genomic data. Its application on pharmacogenomics may fill the gap between genomics and drug response and improve the prediction of drug response in tumors.

## 28. Predict drug sensitivity of cancer cells with pathway activity inference.
**Keywords:** **'Cancer', 'Drug sensitivity', 'Machine learning', 'Pathway activity', 'Pharmacogenomics', 'Precision therapy'**

*Journal: BMC medical genomics* *Publication Date: 2019-Jan-31*

[https://bmcmedgenomics.biomedcentral.com/articles/10.1186/s12920-018-0449-4](https://bmcmedgenomics.biomedcentral.com/articles/10.1186/s12920-018-0449-4){: .btn}

*abstract:* Predicting cellular responses to drugs has been a major challenge for personalized drug therapy regimen. Recent pharmacogenomic studies measured the sensitivities of heterogeneous cell lines to numerous drugs, and provided valuable data resources to develop and validate computational approaches for the prediction of drug responses. Most of current approaches predict drug sensitivity by building prediction models with individual genes, which suffer from low reproducibility due to biologic variability and difficulty to interpret biological relevance of novel gene-drug associations. As an alternative, pathway activity scores derived from gene expression could predict drug response of cancer cells.

## 29. [High definition MRI rectal lymph node aided diagnostic system based on deep neural network].
**Keywords:** **'Artificial intelligence', 'Lymphatic metastasis', 'Magnetic resonance imaging', 'Rectal neoplasms'**

*Journal: Zhonghua wai ke za zhi [Chinese journal of surgery]* *Publication Date: 2019-Feb-01*

[http://journal.yiigle.com/LinkIn.do?linkin_type=pubmed&issn=0529-5815&year=2019&vol=57&issue=2&fpage=108](http://journal.yiigle.com/LinkIn.do?linkin_type=pubmed&issn=0529-5815&year=2019&vol=57&issue=2&fpage=108){: .btn}

*abstract:* 

## 30. [Cold thinking in the boom of artificial intelligence].
**Keywords:** **'Artificial intelligence', 'Decision support systems, clinical', 'Neoplasms'**

*Journal: Zhonghua wai ke za zhi [Chinese journal of surgery]* *Publication Date: 2019-Feb-01*

[http://journal.yiigle.com/LinkIn.do?linkin_type=pubmed&issn=0529-5815&year=2019&vol=57&issue=2&fpage=85](http://journal.yiigle.com/LinkIn.do?linkin_type=pubmed&issn=0529-5815&year=2019&vol=57&issue=2&fpage=85){: .btn}

*abstract:* Artificial intelligence clinical decision-support system is an important direction of artificial intelligence in the medical field. Both international and domestic researchers are exploring the application value of intelligent decision-making system in the field of cancer. But at the same time of the craze, there are still some problems in the intelligent decision-making system. Combining the work of the research groups in this field, this paper explores the current confusions and solutions, and hopes to help clinicians better understand intelligent decision-making. It is believed that with the deepening of the concept and the advancement of technology, intelligent decision-making will become a good help for doctors in the future.

