---
layout: post
title: 2019/02/07 19 Updates (11 - 19)
---
#### 11. Structural connectivity centrality changes mark the path toward Alzheimer's disease.
No LinkOut links available for UID: 30723773

**Authors:** *Peraza Luis R, Díaz-Parra Antonio, Kennion Oliver, Moratal David, Taylor John-Paul, Kaiser Marcus, Bauer Roman*

**Journal:** *Alzheimer's & dementia (Amsterdam, Netherlands)*

*abstract:* The pathophysiological process of Alzheimer's disease is thought to begin years before clinical decline, with evidence suggesting prion-like spreading processes of neurofibrillary tangles and amyloid plaques.

**Keywords:** **"Alzheimer's disease", 'Computational modeling', 'Diffusion MRI', 'Machine learning', 'Network centrality', 'Structural brain connectivity'**

**Publication Date:** *2019-Dec*

#### 12. mlDEEPre: Multi-Functional Enzyme Function Prediction With Hierarchical Multi-Label Deep Learning.
No LinkOut links available for UID: 30723495

**Authors:** *Zou Zhenzhen, Tian Shuye, Gao Xin, Li Yu*

**Journal:** *Frontiers in genetics*

*abstract:* As a great challenge in bioinformatics, enzyme function prediction is a significant step toward designing novel enzymes and diagnosing enzyme-related diseases. Existing studies mainly focus on the mono-functional enzyme function prediction. However, the number of multi-functional enzymes is growing rapidly, which requires novel computational methods to be developed. In this paper, following our previous work, DEEPre, which uses deep learning to annotate mono-functional enzyme's function, we propose a novel method, mlDEEPre, which is designed specifically for predicting the functionalities of multi-functional enzymes. By adopting a novel loss function, associated with the relationship between different labels, and a self-adapted label assigning threshold, mlDEEPre can accurately and efficiently perform multi-functional enzyme prediction. Extensive experiments also show that mlDEEPre can outperform the other methods in predicting whether an enzyme is a mono-functional or a multi-functional enzyme (mono-functional vs. multi-functional), as well as the main class prediction across different criteria. Furthermore, due to the flexibility of mlDEEPre and DEEPre, mlDEEPre can be incorporated into DEEPre seamlessly, which enables the updated DEEPre to handle both mono-functional and multi-functional predictions without human intervention.

**Keywords:** **'EC number', 'deep learning', 'function prediction', 'hierarchical classification', 'multi-functional enzyme', 'multi-label learning'**

**Publication Date:** *2018*

#### 13. Machine Learning of Stem Cell Identities From Single-Cell Expression Data via Regulatory Network Archetypes.
No LinkOut links available for UID: 30723489

**Authors:** *Stumpf Patrick S, MacArthur Ben D*

**Journal:** *Frontiers in genetics*

*abstract:* The molecular regulatory network underlying stem cell pluripotency has been intensively studied, and we now have a reliable ensemble model for the "average" pluripotent cell. However, evidence of significant cell-to-cell variability suggests that the activity of this network varies within individual stem cells, leading to differential processing of environmental signals and variability in cell fates. Here, we adapt a method originally designed for face recognition to infer regulatory network patterns within individual cells from single-cell expression data. Using this method we identify three distinct network configurations in cultured mouse embryonic stem cells-corresponding to naïve and formative pluripotent states and an early primitive endoderm state-and associate these configurations with particular combinations of regulatory network activity archetypes that govern different aspects of the cell's response to environmental stimuli, cell cycle status and core information processing circuitry. These results show how variability in cell identities arise naturally from alterations in underlying regulatory network dynamics and demonstrate how methods from machine learning may be used to better understand single cell biology, and the collective dynamics of cell communities.

**Keywords:** **'eigenface approach', 'machine learning (artificial intelligence)', 'pluripotency stem cells', 'regulatory network', 'single-cell data', 'stem cell'**

**Publication Date:** *2019*

#### 14. The Immune System Computes the State of the Body: Crowd Wisdom, Machine Learning, and Immune Cell Reference Repertoires Help Manage Inflammation.
No LinkOut links available for UID: 30723470

**Authors:** *Cohen Irun R, Efroni Sol*

**Journal:** *Frontiers in immunology*

*abstract:* Here, we outline an overview of the mammalian immune system that updates and extends the classical clonal selection paradigm. Rather than focusing on strict self-not-self discrimination, we propose that the system orchestrates variable inflammatory responses that maintain the body and its symbiosis with the microbiome while eliminating the threat from pathogenic infectious agents and from tumors. The paper makes four points: The immune system classifies healthy and pathologic states of the body-including both self and foreign elements-by deploying individual lymphocytes as cellular computing machines; immune cells transform input signals from the body into an output of specific immune reactions.Rather than independent clonal responses, groups of individually activated immune-system cells co-react in lymphoid organs to make collective decisions through a type of self-organizing swarm intelligence or crowd wisdom.Collective choices by swarms of immune cells, like those of schools of fish, are modified by relatively small numbers of individual regulators responding to shifting conditions-such collective inflammatory responses are dynamically responsive.Self-reactive autoantibody and T-cell receptor (TCR) repertoires shared by healthy individuals function in a biological version of experience-based supervised machine learning. Immune system decisions are primed by formative experience with training sets of self-antigens encountered during lymphocyte development; these initially trained T cell and B cell repertoires form a Wellness Profile that then guides immune responses to test sets of antigens encountered later. This experience-based machine learning strategy is analogous to that deployed by supervised machine-learning algorithms. We propose experiments to test these ideas. This overview of the immune system bears clinical implications for monitoring wellness and for treating autoimmune disease, cancer, and allograft reactions.

**Keywords:** **'T cells', 'autoantibodies', 'autoreactive repertoires', 'immune computation', 'machine learning', 'swarm intelligence'**

**Publication Date:** *2019*

#### 15. The Diagnostic Scope of Sensor-Based Gait Analysis in Atypical Parkinsonism: Further Observations.
No LinkOut links available for UID: 30723450

**Authors:** *Gaßner Heiko, Raccagni Cecilia, Eskofier Bjoern M, Klucken Jochen, Wenning Gregor K*

**Journal:** *Frontiers in neurology*

*abstract:* 

**Keywords:** **'Parkinson disease', 'biosensors', 'gait analysis', 'multisystem atrophy', 'neurologic gait disorders', 'postural balance', 'progressive supranuclear palsy'**

**Publication Date:** *2019*

#### 16. How machine learning could keep dangerous DNA out of terrorists' hands.
No LinkOut links available for UID: 30723344

**Authors:** *Reardon Sara*

**Journal:** *Nature*

*abstract:* 

**Keywords:** **'Synthetic biology', 'Systems biology', 'Virology'**

**Publication Date:** *2019-Feb*

#### 17. DNA methylation profiling reliably distinguishes pulmonary enteric adenocarcinoma from metastatic colorectal cancer.
No LinkOut links available for UID: 30723296

**Authors:** *Jurmeister Philipp, Schöler Anne, Arnold Alexander, Klauschen Frederick, Lenze Dido, Hummel Michael, Schweizer Leonille, Bläker Hendrik, Pfitzner Berit Maria, Mamlouk Soulafa, Sers Christine, Denkert Carsten, Stichel Damian, Frost Nikolaj, Horst David, von Laffert Maximilian, Capper David*

**Journal:** *Modern pathology : an official journal of the United States and Canadian Academy of Pathology, Inc*

*abstract:* Pulmonary enteric adenocarcinoma is a rare non-small cell lung cancer subtype. It is poorly characterized and cannot be distinguished from metastatic colorectal or upper gastrointestinal adenocarcinomas by means of routine pathological methods. As DNA methylation patterns are known to be highly tissue specific, we aimed to develop a methylation-based algorithm to differentiate these entities. To this end, genome-wide methylation profiles of 600 primary pulmonary, colorectal, and upper gastrointestinal adenocarcinomas obtained from The Cancer Genome Atlas and the Gene Expression Omnibus database were used as a reference cohort to train a machine learning algorithm. The resulting classifier correctly classified all samples from a validation cohort consisting of 680 primary pulmonary, colorectal and upper gastrointestinal adenocarcinomas, demonstrating the ability of the algorithm to reliably distinguish these three entities. We then analyzed methylation data of 15 pulmonary enteric adenocarcinomas as well as four pulmonary metastases and four primary colorectal adenocarcinomas with the algorithm. All 15 pulmonary enteric adenocarcinomas were reliably classified as primary pulmonary tumors and all four metastases as well as all four primary colorectal cancer samples were identified as colorectal adenocarcinomas. In a t-distributed stochastic neighbor embedding analysis, the pulmonary enteric adenocarcinoma samples did not form a separate methylation subclass but rather diffusely intermixed with other pulmonary cancers. Additional characterization of the pulmonary enteric adenocarcinoma series using fluorescence in situ hybridization, next-generation sequencing and copy number analysis revealed KRAS mutations in nine of 15 samples (60%) and a high number of structural chromosomal changes. Except for an unusually high rate of chromosome 20 gain (67%), the molecular data was mostly reminiscent of standard pulmonary adenocarcinomas. In conclusion, we provide sound evidence of the pulmonary origin of pulmonary enteric adenocarcinomas and in addition provide a publicly available machine learning-based algorithm to reliably distinguish these tumors from metastatic colorectal cancer.



**Publication Date:** *2019-Feb-05*

#### 18. Genome-wide promoter methylation of hairy cell leukemia.
No LinkOut links available for UID: 30723113

**Authors:** *Arribas Alberto J, Rinaldi Andrea, Chiodin Giorgia, Kwee Ivo, Mensah Afua Adjeiwaa, Cascione Luciano, Rossi Davide, Kanduri Meena, Rosenquist Richard, Zucca Emanuele, Johnson Peter W, Gaidano Gianluca, Oakes Christopher C, Bertoni Francesco, Forconi Francesco*

**Journal:** *Blood advances*

*abstract:* Classic hairy cell leukemia (HCL) is a tumor of mature clonal B cells with unique genetic, morphologic, and phenotypic features. DNA methylation profiling has provided a new tier of investigation to gain insight into the origin and behavior of B-cell malignancies; however, the methylation profile of HCL has not been specifically investigated. DNA methylation profiling was analyzed with the Infinium HumanMethylation27 array in 41 mature B-cell tumors, including 11 HCL, 7 splenic marginal zone lymphomas (SMZLs), and chronic lymphocytic leukemia with an unmutated (n = 7) or mutated (n = 6) immunoglobulin gene heavy chain variable (IGHV) region or using IGHV3-21 (n = 10). Methylation profiles of nontumor B-cell subsets and gene expression profiling data were obtained from public databases. HCL had a methylation signature distinct from each B-cell tumor entity, including the closest entity, SMZL. Comparison with normal B-cell subsets revealed the strongest similarity with postgerminal center (GC) B cells and a clear separation from pre-GC and GC cellular programs. Comparison of the integrated analysis with post-GC B cells revealed significant hypomethylation and overexpression of BCR-TLR-NF-κB and BRAF-MAPK signaling pathways and cell adhesion, as well as hypermethylation and underexpression of cell-differentiation markers and methylated genes in cancer, suggesting regulation of the transformed hairy cells through specific components of the B-cell receptor and the BRAF signaling pathways. Our data identify a specific methylation profile of HCL, which may help to distinguish it from other mature B-cell tumors.



**Publication Date:** *2019-Feb-12*

#### 19. Toxicologic Pathology Forum: Opinion on Integrating Innovative Digital Pathology Tools in the Regulatory Framework.
No LinkOut links available for UID: 30722763

**Authors:** *Gauthier Béatrice E, Gervais Frédéric, Hamm Gregory, O'Shea Donal, Piton Alain, Schumacher Vanessa L*

**Journal:** *Toxicologic pathology*

*abstract:* Digital pathology is defined as the ability to examine digitized microscopic slides and to generate qualitative and quantitative data. The field of digital pathology is rapidly evolving and has the potential to revolutionize toxicologic pathology. Techniques such as automated 2-D image analysis, whole slide imaging, and telepathology are already considered "mature" technologies and have been used for decades in exploratory studies; however, many organizations are reluctant to use digital pathology in regulatory toxicology studies. Innovative technologies using digitized slides including high-content imaging modalities and artificial intelligence are still under development but are increasingly used in toxicologic pathology. While software validation requirements are already described, clear guidance for application of these rules to the digital pathology field are few and the acceptance of these technologies by regulatory authorities remains necessary for successful adoption of digital pathology into the mainstream of toxicologic pathology. This topic was discussed during a roundtable at the 2018 Annual Congress of the French Society of Toxicologic Pathology. This opinion article summarizes the discussion regarding the current questions and challenges on the integration of innovative digital pathology tools within a good laboratory practice framework and is meant to stimulate further discussion among the toxicologic pathology community. This is an opinion article submitted to the Toxicologic Pathology Forum and does not constitute an official position of the Society of Toxicologic Pathology or the journal Toxicologic Pathology. The views expressed in this article are those of the authors and do not necessarily represent the policies, positions, or opinions of their respective agencies and organizations. The Toxicologic Forum is designed to stimulate broad discussion of topics relevant to regulatory issues in Toxicologic pathology. Readers of Toxicologic Pathology are encouraged to send their thoughts on these articles or ideas for new topics to toxicologicpathologyforum@toxpath.org .

**Keywords:** **'GLP', 'artificial intelligence', 'digital pathology', 'regulatory'**

**Publication Date:** *2019-Feb-05*

