---
layout: post
title: 2019/01/31 30 Updates (11 - 20)
---
#### [11. Deep learning for molecular generation.](https://www.future-science.com/doi/full/10.4155/fmc-2018-0358?url_ver=Z39.88-2003&rfr_id=ori:rid:crossref.org&rfr_dat=cr_pub%3dpubmed){: .btn}
**Authors:** *Xu Youjun, Lin Kangjie, Wang Shiwei, Wang Lei, Cai Chenjing, Song Chen, Lai Luhua, Pei Jianfeng*

**Journal:** *Future medicinal chemistry*

*abstract:* De novo drug design aims to generate novel chemical compounds with desirable chemical and pharmacological properties from scratch using computer-based methods. Recently, deep generative neural networks have become a very active research frontier in de novo drug discovery, both in theoretical and in experimental evidence, shedding light on a promising new direction of automatic molecular generation and optimization. In this review, we discussed recent development of deep learning models for molecular generation and summarized them as four different generative architectures with four different optimization strategies. We also discussed future directions of deep generative models for de novo drug design.

**Keywords:** **'  drug design', 'automatic molecular generation', 'deep generative neural networks', 'molecular optimization'**

**Publication Date:** *2019-Jan-30*

#### [12. Prediction of mortality following pediatric heart transplant using machine learning algorithms.](https://doi.org/10.1111/petr.13360){: .btn}
**Authors:** *Miller Rebecca, Tumin Dmitry, Cooper Jennifer, Hayes Don, Tobias Joseph D*

**Journal:** *Pediatric transplantation*

*abstract:* Optimizing transplant candidates' priority for donor organs depends on the accurate assessment of post-transplant outcomes. Due to the complexity of transplantation and the wide range of possible serious complications, recipient outcomes are difficult to predict accurately using conventional multivariable regression. Therefore, we evaluated the utility of 3 ML algorithms for predicting mortality after pediatric HTx.

**Keywords:** **'UNOS', 'heart transplantation', 'machine learning', 'united network for organ sharing'**

**Publication Date:** *2019-Jan-29*

#### [13. Attribute selection and model evaluation for the maternal and paternal imprinted genes in bovine (Bos Taurus) using supervised machine learning algorithms.](https://doi.org/10.1111/jbg.12379){: .btn}
**Authors:** *Karami Keyvan, Zerehdaran Saeed, Javadmanesh Ali, Shariati Mohammad Mahdi, Fallahi Hossien*

**Journal:** *Journal of animal breeding and genetics = Zeitschrift fur Tierzuchtung und Zuchtungsbiologie*

*abstract:* Imprinted genes display biased expression of paternal and maternal alleles in mammals. They are marked through epigenetic process during gametogenesis. Characterization of imprinted genes has expanded our understanding of the regulation and function of genes. In the current study, 22 experimentally validated imprinted genes in bovine (Bos Taurus) were analysed. Several supervised machine learning algorithms and attribute weighting methods were used to find characteristics of different types of imprinted genes and suggest a classification method for finding maternally and paternally expressed genes in bovine. For assessing the best model and comparing attributes in other organisms, we have also conducted a comparative analysis for human and sheep imprinted genes. According to the results of the present study, GC contents 10 and 100 kb upstream, Gly and Gln amino acids, Ile/ATC codon usage, LINE and SINE in 100kbup and length of first intron were significantly different between the maternal and paternal genes in cattle. Considering all species together, we found that GC content 100 kb up, LINE 100 kb up and the frequency of amino acids like Gly, Gln and Met were the most important attributes for identifying the paternal and maternal imprinted genes. These findings could imply conservation pattern in the attributes among these species.

**Keywords:** **'\nBos Taurus\n', 'imprinted genes', 'machine learning', 'maternal and paternal'**

**Publication Date:** *2019-Jan-29*

#### [14. Artifact correction in low-dose dental CT imaging using Wasserstein generative adversarial networks.](https://doi.org/10.1002/mp.13415){: .btn}
**Authors:** *Hu Zhanli, Jiang Changhui, Sun Fengyi, Zhang Qiyang, Ge Yongshuai, Yang Yongfeng, Liu Xin, Zheng Hairong, Liang Dong*

**Journal:** *Medical physics*

*abstract:* In recent years, health risks concerning high-dose X-ray radiation have become a major concern in dental computed tomography (CT) examinations. Therefore, adopting low-dose computed tomography (LDCT) technology has become a major focus in the CT imaging field. One of these LDCT technologies is downsampling data acquisition during low-dose X-ray imaging processes. However, reducing the radiation dose can adversely affect CT image quality by introducing noise and artifacts in the resultant image that can compromise diagnostic information. In this paper, we propose an artifact correction method for downsampling CT reconstruction based on deep learning.

**Keywords:** **'Wasserstein GAN', 'artifact correction', 'deep learning', 'dental CT'**

**Publication Date:** *2019-Jan-29*

#### [15. Inflammatory Biomarkers in Infective Endocarditis: Machine Learning to Predict Mortality.](https://doi.org/10.1111/cei.13266){: .btn}
**Authors:** *Ris Tijmen, Teixeira-Carvalho Andréa, Coelho Rodrigo Matos Pinto, Brandao-de-Resende Camilo, Gomes Matheus Souza, Amaral Laurence Rodrigues, Pinto Pedro Henrique Oliveira Murta, Santos Lorena Junia Souza, Salles Jonathas Teixeira, Roos-Hesselink Jolien, Verkaik Nelianne, Abreu Ferrari Teresa Cristina, Nunes Maria Carmo Pereira*

**Journal:** *Clinical and experimental immunology*

*abstract:* Infective endocarditis (IE) is the cardiac disease with highest rates of mortality. New biomarkers that are able to identify patients at risk for death are required to improve patient management and outcome.

**Keywords:** **'biomarkers', 'cytokines', 'infective endocarditis', 'mortality'**

**Publication Date:** *2019-Jan-29*

#### [16. Prioritizing bona fide bacterial small RNAs with machine learning classifiers.](https://doi.org/10.7717/peerj.6304){: .btn}
**Authors:** *Eppenhof Erik J J, Peña-Castillo Lourdes*

**Journal:** *PeerJ*

*abstract:* Bacterial small (sRNAs) are involved in the control of several cellular processes. Hundreds of putative sRNAs have been identified in many bacterial species through RNA sequencing. The existence of putative sRNAs is usually validated by Northern blot analysis. However, the large amount of novel putative sRNAs reported in the literature makes it impractical to validate each of them in the wet lab. In this work, we applied five machine learning approaches to construct twenty models to discriminate bona fide sRNAs from random genomic sequences in five bacterial species. Sequences were represented using seven features including free energy of their predicted secondary structure, their distances to the closest predicted promoter site and Rho-independent terminator, and their distance to the closest open reading frames (ORFs). To automatically calculate these features, we developed an sRNA Characterization Pipeline (sRNACharP). All seven features used in the classification task contributed positively to the performance of the predictive models. The best performing model obtained a median precision of 100% at 10% recall and of 64% at 40% recall across all five bacterial species, and it outperformed previous published approaches on two benchmark datasets in terms of precision and recall. Our results indicate that even though there is limited sRNA sequence conservation across different bacterial species, there are intrinsic features in the genomic context of sRNAs that are conserved across taxa. We show that these features are utilized by machine learning approaches to learn a species-independent model to prioritize bona fide bacterial sRNAs.

**Keywords:** **'Bacterial small RNAs', 'Machine learning', 'Multispecies evaluation', 'Random forest', 'sRNA characterization', 'sRNA prediction', 'sRNA prioritization', 'sRNACharP', 'sRNARanking'**

**Publication Date:** *2019*

#### [17. Multispectral imaging for presymptomatic analysis of light leaf spot in oilseed rape.](https://plantmethods.biomedcentral.com/articles/10.1186/s13007-019-0389-9){: .btn}
**Authors:** *Veys Charles, Chatziavgerinos Fokion, AlSuwaidi Ali, Hibbert James, Hansen Mark, Bernotas Gytis, Smith Melvyn, Yin Hujun, Rolfe Stephen, Grieve Bruce*

**Journal:** *Plant methods*

*abstract:* The use of spectral imaging within the plant phenotyping and breeding community has been increasing due its utility as a non-invasive diagnostic tool. However, there is a lack of imaging systems targeted specifically at plant science duties, resulting in low precision for canopy-scale measurements. This study trials a prototype multispectral system designed specifically for plant studies and looks at its use as an early detection system for visually asymptomatic disease phases, in this case 

**Keywords:** **'Disease detection', 'Light leaf spot', 'Machine learning', 'Multispectral', 'Novelty detection', 'Oilseed rape', 'Orientation effects', 'Photometric stereo', 'Preprocessing', 'Support vector machine'**

**Publication Date:** *2019*

#### [18. FLOating-Window Projective Separator (FloWPS): A Data Trimming Tool for Support Vector Machines (SVM) to Improve Robustness of the Classifier.](https://dx.doi.org/10.3389/fgene.2018.00717){: .btn}
**Authors:** *Tkachev Victor, Sorokin Maxim, Mescheryakov Artem, Simonov Alexander, Garazha Andrew, Buzdin Anton, Muchnik Ilya, Borisov Nicolas*

**Journal:** *Frontiers in genetics*

*abstract:* Here, we propose a heuristic technique of data trimming for SVM termed 

**Keywords:** **'bioinformatics', 'gene expression', 'machine learning', 'oncology', 'personalized medicine', 'support vector machines'**

**Publication Date:** *2018*

#### [19. Guidance of Navigating Honeybees by Learned Elongated Ground Structures.](https://dx.doi.org/10.3389/fnbeh.2018.00322){: .btn}
**Authors:** *Menzel Randolf, Tison Lea, Fischer-Nakai Johannes, Cheeseman James, Balbuena Maria Sol, Chen Xiuxian, Landgraf Tim, Petrasch Julian, Polster Johannes, Greggers Uwe*

**Journal:** *Frontiers in behavioral neuroscience*

*abstract:* Elongated landscape features like forest edges, rivers, roads or boundaries of fields are particularly salient landmarks for navigating animals. Here, we ask how honeybees learn such structures and how they are used during their homing flights after being released at an unexpected location (catch-and-release paradigm). The experiments were performed in two landscapes that differed with respect to their overall structure: a rather feature-less landscape, and one rich in close and far distant landmarks. We tested three different forms of learning: learning during orientation flights, learning during training to a feeding site, and learning during homing flights after release at an unexpected site within the explored area. We found that bees use elongated ground structures, e.g., a field boundary separating two pastures close to the hive (Experiment 1), an irrigation channel (Experiment 2), a hedgerow along which the bees were trained (Experiment 3), a gravel road close to the hive and the feeder (Experiment 4), a path along an irrigation channel with its vegetation close to the feeder (Experiment 5) and a gravel road along which bees performed their homing flights (Experiment 6). Discrimination and generalization between the learned linear landmarks and similar ones in the test area depend on their object properties (irrigation channel, gravel road, hedgerow) and their compass orientation. We conclude that elongated ground structures are embedded into multiple landscape features indicating that memory of these linear structures is one component of bee navigation. Elongated structures interact and compete with other references. Object identification is an important part of this process. The objects are characterized not only by their appearance but also by their alignment in the compass. Their salience is highest if both components are close to what had been learned. High similarity in appearance can compensate for (partial) compass misalignment, and vice versa.

**Keywords:** **'compass alignment', 'ground structures', 'guiding landmarks', 'navigation', 'object recognition', 'sun compass'**

**Publication Date:** *2018*

#### [20. A Fast and Refined Cancer Regions Segmentation Framework in Whole-slide Breast Pathological Images.](http://dx.doi.org/10.1038/s41598-018-37492-9){: .btn}
**Authors:** *Guo Zichao, Liu Hong, Ni Haomiao, Wang Xiangdong, Su Mingming, Guo Wei, Wang Kuansong, Jiang Taijiao, Qian Yueliang*

**Journal:** *Scientific reports*

*abstract:* Supervised learning methods are commonly applied in medical image analysis. However, the success of these approaches is highly dependent on the availability of large manually detailed annotated dataset. Thus an automatic refined segmentation of whole-slide image (WSI) is significant to alleviate the annotation workload of pathologists. But most of the current ways can only output a rough prediction of lesion areas and consume much time in each slide. In this paper, we propose a fast and refined cancer regions segmentation framework v3_DCNN, which first preselects tumor regions using a classification model Inception-v3 and then employs a semantic segmentation model DCNN for refined segmentation. Our framework can generate a dense likelihood heatmap with the 1/8 side of original WSI in 11.5 minutes on the Camelyon16 dataset, which saves more than one hour for each WSI compared with the initial DCNN model. Experimental results show that our approach achieves a higher FROC score 83.5% with the champion's method of Camelyon16 challenge 80.7%. Based on v3 DCNN model, we further automatically produce heatmap of WSI and extract polygons of lesion regions for doctors, which is very helpful for their pathological diagnosis, detailed annotation and thus contributes to developing a more powerful deep learning model.



**Publication Date:** *2019-Jan-29*

