---
layout: post
title: 2019/01/31 36 Updates (11 - 20)
---
#### [11. Reduction of recruitment costs in preclinical AD trials: validation of automatic pre-screening algorithm for brain amyloidosis.](http://journals.sagepub.com/doi/full/10.1177/0962280218823036?url_ver=Z39.88-2003&rfr_id=ori:rid:crossref.org&rfr_dat=cr_pub%3dpubmed){: .btn}
**Authors:** *Ansart Manon, Epelbaum Stéphane, Gagliardi Geoffroy, Colliot Olivier, Dormont Didier, Dubois Bruno, Hampel Harald, Durrleman Stanley*

**Journal:** *Statistical methods in medical research*

*abstract:* We propose a method for recruiting asymptomatic Amyloid positive individuals in clinical trials, using a two-step process. We first select during a pre-screening phase a subset of individuals which are more likely to be amyloid positive based on the automatic analysis of data acquired during routine clinical practice, before doing a confirmatory PET-scan to these selected individuals only. This method leads to an increased number of recruitments and to a reduced number of PET-scans, resulting in a decrease in overall recruitment costs. We validate our method on three different cohorts, and consider five different classification algorithms for the pre-screening phase. We show that the best results are obtained using solely cognitive, genetic and socio-demographic features, as the slight increased performance when using MRI or longitudinal data is balanced by the cost increase they induce. We show that the proposed method generalizes well when tested on an independent cohort, and that the characteristics of the selected set of individuals are identical to the characteristics of a population selected in a standard way. The proposed approach shows how Machine Learning can be used effectively in practice to optimize recruitment costs in clinical trials.

**Keywords:** **'Alzheimer’s disease', 'Pre-screening for clinical trials', 'Random Forest', 'amyloidosis', 'classification', 'longitudinal data', 'recruitment costs'**

**Publication Date:** *2019-Jan-30*

#### [12. Deep learning for molecular generation.](https://www.future-science.com/doi/full/10.4155/fmc-2018-0358?url_ver=Z39.88-2003&rfr_id=ori:rid:crossref.org&rfr_dat=cr_pub%3dpubmed){: .btn}
**Authors:** *Xu Youjun, Lin Kangjie, Wang Shiwei, Wang Lei, Cai Chenjing, Song Chen, Lai Luhua, Pei Jianfeng*

**Journal:** *Future medicinal chemistry*

*abstract:* De novo drug design aims to generate novel chemical compounds with desirable chemical and pharmacological properties from scratch using computer-based methods. Recently, deep generative neural networks have become a very active research frontier in de novo drug discovery, both in theoretical and in experimental evidence, shedding light on a promising new direction of automatic molecular generation and optimization. In this review, we discussed recent development of deep learning models for molecular generation and summarized them as four different generative architectures with four different optimization strategies. We also discussed future directions of deep generative models for de novo drug design.

**Keywords:** **'  drug design', 'automatic molecular generation', 'deep generative neural networks', 'molecular optimization'**

**Publication Date:** *2019-Jan-30*

#### [13. Machine learning in radiology.](https://doi.org/10.1111/1754-9485.12853){: .btn}
**Authors:** *Brotchie Peter*

**Journal:** *Journal of medical imaging and radiation oncology*

*abstract:* 



**Publication Date:** *2019-Feb*

#### [14. Prediction of mortality following pediatric heart transplant using machine learning algorithms.](https://doi.org/10.1111/petr.13360){: .btn}
**Authors:** *Miller Rebecca, Tumin Dmitry, Cooper Jennifer, Hayes Don, Tobias Joseph D*

**Journal:** *Pediatric transplantation*

*abstract:* Optimizing transplant candidates' priority for donor organs depends on the accurate assessment of post-transplant outcomes. Due to the complexity of transplantation and the wide range of possible serious complications, recipient outcomes are difficult to predict accurately using conventional multivariable regression. Therefore, we evaluated the utility of 3 ML algorithms for predicting mortality after pediatric HTx.

**Keywords:** **'UNOS', 'heart transplantation', 'machine learning', 'united network for organ sharing'**

**Publication Date:** *2019-Jan-29*

#### [15. Attribute selection and model evaluation for the maternal and paternal imprinted genes in bovine (Bos Taurus) using supervised machine learning algorithms.](https://doi.org/10.1111/jbg.12379){: .btn}
**Authors:** *Karami Keyvan, Zerehdaran Saeed, Javadmanesh Ali, Shariati Mohammad Mahdi, Fallahi Hossien*

**Journal:** *Journal of animal breeding and genetics = Zeitschrift fur Tierzuchtung und Zuchtungsbiologie*

*abstract:* Imprinted genes display biased expression of paternal and maternal alleles in mammals. They are marked through epigenetic process during gametogenesis. Characterization of imprinted genes has expanded our understanding of the regulation and function of genes. In the current study, 22 experimentally validated imprinted genes in bovine (Bos Taurus) were analysed. Several supervised machine learning algorithms and attribute weighting methods were used to find characteristics of different types of imprinted genes and suggest a classification method for finding maternally and paternally expressed genes in bovine. For assessing the best model and comparing attributes in other organisms, we have also conducted a comparative analysis for human and sheep imprinted genes. According to the results of the present study, GC contents 10 and 100 kb upstream, Gly and Gln amino acids, Ile/ATC codon usage, LINE and SINE in 100kbup and length of first intron were significantly different between the maternal and paternal genes in cattle. Considering all species together, we found that GC content 100 kb up, LINE 100 kb up and the frequency of amino acids like Gly, Gln and Met were the most important attributes for identifying the paternal and maternal imprinted genes. These findings could imply conservation pattern in the attributes among these species.

**Keywords:** **'\nBos Taurus\n', 'imprinted genes', 'machine learning', 'maternal and paternal'**

**Publication Date:** *2019-Jan-29*

#### [16. Artifact correction in low-dose dental CT imaging using Wasserstein generative adversarial networks.](https://doi.org/10.1002/mp.13415){: .btn}
**Authors:** *Hu Zhanli, Jiang Changhui, Sun Fengyi, Zhang Qiyang, Ge Yongshuai, Yang Yongfeng, Liu Xin, Zheng Hairong, Liang Dong*

**Journal:** *Medical physics*

*abstract:* In recent years, health risks concerning high-dose X-ray radiation have become a major concern in dental computed tomography (CT) examinations. Therefore, adopting low-dose computed tomography (LDCT) technology has become a major focus in the CT imaging field. One of these LDCT technologies is downsampling data acquisition during low-dose X-ray imaging processes. However, reducing the radiation dose can adversely affect CT image quality by introducing noise and artifacts in the resultant image that can compromise diagnostic information. In this paper, we propose an artifact correction method for downsampling CT reconstruction based on deep learning.

**Keywords:** **'Wasserstein GAN', 'artifact correction', 'deep learning', 'dental CT'**

**Publication Date:** *2019-Jan-29*

#### [17. Inflammatory Biomarkers in Infective Endocarditis: Machine Learning to Predict Mortality.](https://doi.org/10.1111/cei.13266){: .btn}
**Authors:** *Ris Tijmen, Teixeira-Carvalho Andréa, Coelho Rodrigo Matos Pinto, Brandao-de-Resende Camilo, Gomes Matheus Souza, Amaral Laurence Rodrigues, Pinto Pedro Henrique Oliveira Murta, Santos Lorena Junia Souza, Salles Jonathas Teixeira, Roos-Hesselink Jolien, Verkaik Nelianne, Abreu Ferrari Teresa Cristina, Nunes Maria Carmo Pereira*

**Journal:** *Clinical and experimental immunology*

*abstract:* Infective endocarditis (IE) is the cardiac disease with highest rates of mortality. New biomarkers that are able to identify patients at risk for death are required to improve patient management and outcome.

**Keywords:** **'biomarkers', 'cytokines', 'infective endocarditis', 'mortality'**

**Publication Date:** *2019-Jan-29*

#### [18. Prioritizing bona fide bacterial small RNAs with machine learning classifiers.](https://doi.org/10.7717/peerj.6304){: .btn}
**Authors:** *Eppenhof Erik J J, Peña-Castillo Lourdes*

**Journal:** *PeerJ*

*abstract:* Bacterial small (sRNAs) are involved in the control of several cellular processes. Hundreds of putative sRNAs have been identified in many bacterial species through RNA sequencing. The existence of putative sRNAs is usually validated by Northern blot analysis. However, the large amount of novel putative sRNAs reported in the literature makes it impractical to validate each of them in the wet lab. In this work, we applied five machine learning approaches to construct twenty models to discriminate bona fide sRNAs from random genomic sequences in five bacterial species. Sequences were represented using seven features including free energy of their predicted secondary structure, their distances to the closest predicted promoter site and Rho-independent terminator, and their distance to the closest open reading frames (ORFs). To automatically calculate these features, we developed an sRNA Characterization Pipeline (sRNACharP). All seven features used in the classification task contributed positively to the performance of the predictive models. The best performing model obtained a median precision of 100% at 10% recall and of 64% at 40% recall across all five bacterial species, and it outperformed previous published approaches on two benchmark datasets in terms of precision and recall. Our results indicate that even though there is limited sRNA sequence conservation across different bacterial species, there are intrinsic features in the genomic context of sRNAs that are conserved across taxa. We show that these features are utilized by machine learning approaches to learn a species-independent model to prioritize bona fide bacterial sRNAs.

**Keywords:** **'Bacterial small RNAs', 'Machine learning', 'Multispecies evaluation', 'Random forest', 'sRNA characterization', 'sRNA prediction', 'sRNA prioritization', 'sRNACharP', 'sRNARanking'**

**Publication Date:** *2019*

#### [19. Multispectral imaging for presymptomatic analysis of light leaf spot in oilseed rape.](https://plantmethods.biomedcentral.com/articles/10.1186/s13007-019-0389-9){: .btn}
**Authors:** *Veys Charles, Chatziavgerinos Fokion, AlSuwaidi Ali, Hibbert James, Hansen Mark, Bernotas Gytis, Smith Melvyn, Yin Hujun, Rolfe Stephen, Grieve Bruce*

**Journal:** *Plant methods*

*abstract:* The use of spectral imaging within the plant phenotyping and breeding community has been increasing due its utility as a non-invasive diagnostic tool. However, there is a lack of imaging systems targeted specifically at plant science duties, resulting in low precision for canopy-scale measurements. This study trials a prototype multispectral system designed specifically for plant studies and looks at its use as an early detection system for visually asymptomatic disease phases, in this case 

**Keywords:** **'Disease detection', 'Light leaf spot', 'Machine learning', 'Multispectral', 'Novelty detection', 'Oilseed rape', 'Orientation effects', 'Photometric stereo', 'Preprocessing', 'Support vector machine'**

**Publication Date:** *2019*

#### 20. Ontogeny of vocal rhythms in harbor seal pups: an exploratory study.
No LinkOut links available for UID: 30697246

**Authors:** *Ravignani Andrea, Kello Christopher T, de Reus Koen, Kotz Sonja A, Dalla Bella Simone, Méndez-Aróstegui Margarita, Rapado-Tamarit Beatriz, Rubio-Garcia Ana, de Boer Bart*

**Journal:** *Current zoology*

*abstract:* Puppyhood is a very active social and vocal period in a harbor seal's life 

**Keywords:** **'bioacoustics', 'pinnipeds', 'rhythm', 'timing', 'vocal development'**

**Publication Date:** *2019-Feb*

