---
layout: post
title: "2019/01/31 36 Updates"
---
## 1. Estimating blood pressure trends and the nocturnal dip from photoplethysmography.
**Keywords:** **'Ambulatory blood pressure', 'Free-living protocol', 'Neural networks', 'Photoplethysmography'**

*Journal: Physiological measurement* *Publication Date: 2019-Jan-30*

[https://doi.org/10.1088/1361-6579/ab030e](https://doi.org/10.1088/1361-6579/ab030e){: .btn}

*abstract:* &lt;b&gt;Objective:&lt;/b&gt; Evaluate a method for the estimation of the nocturnal systolic blood pressure (SBP) dip from 24-hour blood pressure (BP) trends using a wrist-worn photoplethysmography (PPG) sensor and a deep neural network in free-living individuals, comparing the deep neural network to traditional machine learning and non-machine learning baselines. &#13; &#13; &lt;b&gt;Approach:&lt;/b&gt; A wrist-worn PPG sensor was worn by 106 healthy individuals for 226 days during which 5111 reference values for BP were obtained with a 24-hour ambulatory BP monitor and matched with the PPG sensor data. Features based on heart rate variability and pulse morphology were extracted from the PPG waveforms. Long- and short term memory (LSTM) networks, dense networks, random forests and linear regression models were trained and evaluated in their capability of tracking trends in BP, as well as the estimation of the SBP dip. &#13; &#13; &lt;b&gt;Main results:&lt;/b&gt; Best performance for estimating the SBP dip were obtained with a deep LSTM neural network with a root mean squared error (RMSE) of 3.12±2.20 mmHg and a correlation of 0.69 (p=3·10&lt;sup&gt;4&lt;/sup&gt;). This dip was derived from trend estimates of BP which had an RMSE of 8.22±1.49 mmHg for systolic and 6.55±1.39 mmHg for diastolic BP (DBP). While other models had similar performance for the tracking of relative BP, they did not perform as well as the LSTM for the SBP dip. &#13; &#13; &lt;b&gt;Significance:&lt;/b&gt; The work provides first evidence for the unobtrusive estimation of the nocturnal SBP dip, a highly prognostic clinical parameter. It is also the first to evaluate unobtrusive BP measurement in a large data set of unconstrained 24- hour measurements in free-living individuals and provides evidence for the utility of LSTM models in this domain.

## 2. Predicting forced vital capacity (FVC) using support vector regression (SVR).
**Keywords:** **'Chronic Obstructive Pulmonary Disease', 'forced vital capacity', 'machine learning', 'prediction'**

*Journal: Physiological measurement* *Publication Date: 2019-Jan-30*

[https://doi.org/10.1088/1361-6579/ab031c](https://doi.org/10.1088/1361-6579/ab031c){: .btn}

*abstract:* &#13; Objectives: Spirometry, as the golden approach in the diagnosis of chronic obstructive pulmonary disease (COPD), has strict end-of-test (EOT) criteria (e.g., complete exhalation), which cannot be met by patients with compromised health situations. Thus, significant parameters measured by spirometry such as FVC has limited accuracies. In order to address this issue, the present study was aimed to develop models based on support vector regression to predict values of FVC under the condition that the EOT criteria were not fully met.&#13; Approach: The prediction models for the quantification of FVC were developed based on SVR. 354 subjects conducted conventional spirometry (CS) and the resulting data of forced expiratory volumes in one second (FEV1), peak expiratory flow(PEF), age and gender were used as input features and the resulting values of the FVC were used as the target feature in the prediction models. After that, three prediction models (Mixed Model, Normal Model and Abnormal Model) were established according to the criterion in the diagnosis of COPD that a postbronchodilator FEV1/FVC lower than 0.70. Then 35 subjects were recruited to be tested with both CS and low-degree-of EOT criteria spirometry (LDCS), which did not fully met the EOT criteria of CS. In LDCS, subjects were allowed to terminate the procedures based on their own wills at any time after the technicians assumed that both acceptable values of FEV1 and PEF were obtained. Quantified values of FVC derived from both CS and LDCS were compared to validate the performances of the developed prediction models.&#13; Main results: The FVC prediction performance of the Normal Model and Abnormal Model were better than that of the Mixed Model. The RMSE are lower than 0.35L and the accuracy are higher to 95%. One tailed t test results demonstrate that the absolute differences of the measured and predicted values are not significantly different from 0.15L for both Abnormal Model and Normal Model.&#13; Significance: Our study shows the possibility of predicting FVC with acceptable precision in cases of the EOT criteria of spirometry were not fully met, which can be beneficial for patients who cannot or did not make full exhalation in spirometry.&#13.

## 3. Correction: Biomarkers of erosive arthritis in systemic lupus erythematosus: Application of machine learning models.


*Journal: PloS one* *Publication Date: 2019*

[http://dx.plos.org/10.1371/journal.pone.0211791](http://dx.plos.org/10.1371/journal.pone.0211791){: .btn}

*abstract:* [This corrects the article DOI: 10.1371/journal.pone.0207926.].

## 4. Automatic Disease Annotation From Radiology Reports Using Artificial Intelligence Implemented by a Recurrent Neural Network.
**Keywords:** **'automatic annotation', 'deep learning', 'natural language processing', 'radiology reports', 'recurrent neural network'**

*Journal: AJR. American journal of roentgenology* *Publication Date: 2019-Jan-30*

[http://www.ajronline.org/doi/full/10.2214/AJR.18.19869](http://www.ajronline.org/doi/full/10.2214/AJR.18.19869){: .btn}

*abstract:* Radiology reports are rich resources for biomedical researchers. Before utilization of radiology reports, experts must manually review these reports to identify the categories. In fact, automatically categorizing electronic medical record (EMR) text with key annotation is difficult because it has a free-text format. To address these problems, we developed an automated system for disease annotation.

## 5. ELM-MHC: An improved MHC Identification method with Extreme Learning Machine Algorithm.


*Journal: Journal of proteome research* *Publication Date: 2019-Jan-30*

[https://dx.doi.org/10.1021/acs.jproteome.9b00012](https://dx.doi.org/10.1021/acs.jproteome.9b00012){: .btn}

*abstract:* The major histocompatibility complex (MHC) is a term for all gene groups of major histocompatibility antigen. It binds to peptide chains derived from pathogens and displays pathogens on the cell surface to facilitate T-cell recognition and perform a series of immune function. MHC molecules are critical in transplantation, autoimmunity, infection and tumor immunotherapy. Combining machine learning algorithms and making full use of bioinformatics analysis technology, more accurate recognition of MHC is an important task. The paper proposed a new MHC recognition method compared with traditional biological methods. And used the built classifier to classify and identify MHC I and MHC II. The classifier used the SVMProt 188D, bag-of-ngrams (BonG) and information theory (IT) mixed feature representation methods, and used the extreme learning machine (ELM), which selects lin-kernel as the activation function and used ten-fold cross-validation and the independent test set validation verify the accuracy of the constructed classifier, and simultaneously identify the MHC, and identify the MHC I and MHC II respectively. Through the ten-fold cross-validation, the proposed algorithm obtained 91.66% accuracy when identifying MHC, and 94.442% accuracy when identifying MHC categories. Furthermore, an online identification website named ELM-MHC was constructed with the following URL: http://server.malab.cn/ELM-MHC/.

## 6. Developing Machine Learning Models for Behavioral Coding.


*Journal: Journal of pediatric psychology* *Publication Date: 2019-Jan-29*

[https://academic.oup.com/jpepsy/article-lookup/doi/10.1093/jpepsy/jsy113](https://academic.oup.com/jpepsy/article-lookup/doi/10.1093/jpepsy/jsy113){: .btn}

*abstract:* The goal of this research is to develop a machine learning supervised classification model to automatically code clinical encounter transcripts using a behavioral code scheme.

## 7. Predicting runtimes of bioinformatics tools based on historical data: Five years of Galaxy usage.


*Journal: Bioinformatics (Oxford, England)* *Publication Date: 2019-Jan-30*

[https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/btz054](https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/btz054){: .btn}

*abstract:* One of the many technical challenges that arises when scheduling bioinformatics analyses at scale is determining the appropriate amount of memory and processing resources. Both over- and under-allocation leads to an inefficient use of computational infrastructure. Over allocation locks resources that could otherwise be used for other analyses. Under allocation causes job failure and requires analyses to be repeated with a larger memory or runtime allowance. We address this challenge by using a historical dataset of bioinformatics analyses run on the Galaxy platform to demonstrate the feasibility of an online service for resource requirement estimation.

## 8. Consumer Health Search on the Web: Study of Web Page Understandability and Its Integration in Ranking Algorithms.
**Keywords:** **'comprehension', 'literacy', 'machine learning', 'patients', 'readability'**

*Journal: Journal of medical Internet research* *Publication Date: 2019-Jan-30*

[http://www.jmir.org/2019/1/e10986/](http://www.jmir.org/2019/1/e10986/){: .btn}

*abstract:* Understandability plays a key role in ensuring that people accessing health information are capable of gaining insights that can assist them with their health concerns and choices. The access to unclear or misleading information has been shown to negatively impact the health decisions of the general public.

## 9. Interrogation of eukaryotic stop codon readthrough signals by in vitro RNA selection.


*Journal: Biochemistry* *Publication Date: 2019-Jan-30*

[https://dx.doi.org/10.1021/acs.biochem.8b01280](https://dx.doi.org/10.1021/acs.biochem.8b01280){: .btn}

*abstract:* RNA signals located downstream of stop codons in eukaryotic mRNAs can stimulate high levels of translational readthrough (RT) by the ribosome, thereby giving rise to functionally distinct c-terminally extended protein products. Although many RT events have been previously discovered in Nature, a broader description of the stimulatory RNA signals would help to identify new reprogramming events in eukaryotic genes and provide insights into the molecular mechanisms of readthrough. Here, we explore the RNA reprogramming landscape by performing in vitro translation selections to enrich RNA readthrough signals de novo from a starting randomized library comprising >10

## 10. Constructing Prediction Model of Lung Cancer Recurrence Risk Using Gene Function Clustering and Machine Learning.
**Keywords:** **'Functional cluster', 'gene expression', 'machine learning', 'predict model', 'recurrent', 'risk model.'**

*Journal: Combinatorial chemistry & high throughput screening* *Publication Date: 2019-Jan-28*



*abstract:* A significant proportion of patients with early non-small cell lung cancer (NSCLC) can be cured by surgery. The distant metastasis of tumors is the most common cause of treatment failure. Precisely predicting the likelihood that a patient develops distant metastatic risk will help identify patients who can further intervene, such as conventional adjuvant chemotherapy or experimental drugs.

## 11. Reduction of recruitment costs in preclinical AD trials: validation of automatic pre-screening algorithm for brain amyloidosis.
**Keywords:** **'Alzheimer’s disease', 'Pre-screening for clinical trials', 'Random Forest', 'amyloidosis', 'classification', 'longitudinal data', 'recruitment costs'**

*Journal: Statistical methods in medical research* *Publication Date: 2019-Jan-30*

[http://journals.sagepub.com/doi/full/10.1177/0962280218823036?url_ver=Z39.88-2003&rfr_id=ori:rid:crossref.org&rfr_dat=cr_pub%3dpubmed](http://journals.sagepub.com/doi/full/10.1177/0962280218823036?url_ver=Z39.88-2003&rfr_id=ori:rid:crossref.org&rfr_dat=cr_pub%3dpubmed){: .btn}

*abstract:* We propose a method for recruiting asymptomatic Amyloid positive individuals in clinical trials, using a two-step process. We first select during a pre-screening phase a subset of individuals which are more likely to be amyloid positive based on the automatic analysis of data acquired during routine clinical practice, before doing a confirmatory PET-scan to these selected individuals only. This method leads to an increased number of recruitments and to a reduced number of PET-scans, resulting in a decrease in overall recruitment costs. We validate our method on three different cohorts, and consider five different classification algorithms for the pre-screening phase. We show that the best results are obtained using solely cognitive, genetic and socio-demographic features, as the slight increased performance when using MRI or longitudinal data is balanced by the cost increase they induce. We show that the proposed method generalizes well when tested on an independent cohort, and that the characteristics of the selected set of individuals are identical to the characteristics of a population selected in a standard way. The proposed approach shows how Machine Learning can be used effectively in practice to optimize recruitment costs in clinical trials.

## 12. Deep learning for molecular generation.
**Keywords:** **'  drug design', 'automatic molecular generation', 'deep generative neural networks', 'molecular optimization'**

*Journal: Future medicinal chemistry* *Publication Date: 2019-Jan-30*

[https://www.future-science.com/doi/full/10.4155/fmc-2018-0358?url_ver=Z39.88-2003&rfr_id=ori:rid:crossref.org&rfr_dat=cr_pub%3dpubmed](https://www.future-science.com/doi/full/10.4155/fmc-2018-0358?url_ver=Z39.88-2003&rfr_id=ori:rid:crossref.org&rfr_dat=cr_pub%3dpubmed){: .btn}

*abstract:* De novo drug design aims to generate novel chemical compounds with desirable chemical and pharmacological properties from scratch using computer-based methods. Recently, deep generative neural networks have become a very active research frontier in de novo drug discovery, both in theoretical and in experimental evidence, shedding light on a promising new direction of automatic molecular generation and optimization. In this review, we discussed recent development of deep learning models for molecular generation and summarized them as four different generative architectures with four different optimization strategies. We also discussed future directions of deep generative models for de novo drug design.

## 13. Machine learning in radiology.


*Journal: Journal of medical imaging and radiation oncology* *Publication Date: 2019-Jan-30*

[https://doi.org/10.1111/1754-9485.12853](https://doi.org/10.1111/1754-9485.12853){: .btn}

*abstract:* 

## 14. Prediction of mortality following pediatric heart transplant using machine learning algorithms.
**Keywords:** **'UNOS', 'heart transplantation', 'machine learning', 'united network for organ sharing'**

*Journal: Pediatric transplantation* *Publication Date: 2019-Jan-29*

[https://doi.org/10.1111/petr.13360](https://doi.org/10.1111/petr.13360){: .btn}

*abstract:* Optimizing transplant candidates' priority for donor organs depends on the accurate assessment of post-transplant outcomes. Due to the complexity of transplantation and the wide range of possible serious complications, recipient outcomes are difficult to predict accurately using conventional multivariable regression. Therefore, we evaluated the utility of 3 ML algorithms for predicting mortality after pediatric HTx.

## 15. Attribute selection and model evaluation for the maternal and paternal imprinted genes in bovine (Bos Taurus) using supervised machine learning algorithms.
**Keywords:** **'\nBos Taurus\n', 'imprinted genes', 'machine learning', 'maternal and paternal'**

*Journal: Journal of animal breeding and genetics = Zeitschrift fur Tierzuchtung und Zuchtungsbiologie* *Publication Date: 2019-Jan-29*

[https://doi.org/10.1111/jbg.12379](https://doi.org/10.1111/jbg.12379){: .btn}

*abstract:* Imprinted genes display biased expression of paternal and maternal alleles in mammals. They are marked through epigenetic process during gametogenesis. Characterization of imprinted genes has expanded our understanding of the regulation and function of genes. In the current study, 22 experimentally validated imprinted genes in bovine (Bos Taurus) were analysed. Several supervised machine learning algorithms and attribute weighting methods were used to find characteristics of different types of imprinted genes and suggest a classification method for finding maternally and paternally expressed genes in bovine. For assessing the best model and comparing attributes in other organisms, we have also conducted a comparative analysis for human and sheep imprinted genes. According to the results of the present study, GC contents 10 and 100 kb upstream, Gly and Gln amino acids, Ile/ATC codon usage, LINE and SINE in 100kbup and length of first intron were significantly different between the maternal and paternal genes in cattle. Considering all species together, we found that GC content 100 kb up, LINE 100 kb up and the frequency of amino acids like Gly, Gln and Met were the most important attributes for identifying the paternal and maternal imprinted genes. These findings could imply conservation pattern in the attributes among these species.

## 16. Artifact correction in low-dose dental CT imaging using Wasserstein generative adversarial networks.
**Keywords:** **'Wasserstein GAN', 'artifact correction', 'deep learning', 'dental CT'**

*Journal: Medical physics* *Publication Date: 2019-Jan-29*

[https://doi.org/10.1002/mp.13415](https://doi.org/10.1002/mp.13415){: .btn}

*abstract:* In recent years, health risks concerning high-dose X-ray radiation have become a major concern in dental computed tomography (CT) examinations. Therefore, adopting low-dose computed tomography (LDCT) technology has become a major focus in the CT imaging field. One of these LDCT technologies is downsampling data acquisition during low-dose X-ray imaging processes. However, reducing the radiation dose can adversely affect CT image quality by introducing noise and artifacts in the resultant image that can compromise diagnostic information. In this paper, we propose an artifact correction method for downsampling CT reconstruction based on deep learning.

## 17. Inflammatory Biomarkers in Infective Endocarditis: Machine Learning to Predict Mortality.
**Keywords:** **'biomarkers', 'cytokines', 'infective endocarditis', 'mortality'**

*Journal: Clinical and experimental immunology* *Publication Date: 2019-Jan-29*

[https://doi.org/10.1111/cei.13266](https://doi.org/10.1111/cei.13266){: .btn}

*abstract:* Infective endocarditis (IE) is the cardiac disease with highest rates of mortality. New biomarkers that are able to identify patients at risk for death are required to improve patient management and outcome.

## 18. Prioritizing bona fide bacterial small RNAs with machine learning classifiers.
**Keywords:** **'Bacterial small RNAs', 'Machine learning', 'Multispecies evaluation', 'Random forest', 'sRNA characterization', 'sRNA prediction', 'sRNA prioritization', 'sRNACharP', 'sRNARanking'**

*Journal: PeerJ* *Publication Date: 2019*

[https://doi.org/10.7717/peerj.6304](https://doi.org/10.7717/peerj.6304){: .btn}

*abstract:* Bacterial small (sRNAs) are involved in the control of several cellular processes. Hundreds of putative sRNAs have been identified in many bacterial species through RNA sequencing. The existence of putative sRNAs is usually validated by Northern blot analysis. However, the large amount of novel putative sRNAs reported in the literature makes it impractical to validate each of them in the wet lab. In this work, we applied five machine learning approaches to construct twenty models to discriminate bona fide sRNAs from random genomic sequences in five bacterial species. Sequences were represented using seven features including free energy of their predicted secondary structure, their distances to the closest predicted promoter site and Rho-independent terminator, and their distance to the closest open reading frames (ORFs). To automatically calculate these features, we developed an sRNA Characterization Pipeline (sRNACharP). All seven features used in the classification task contributed positively to the performance of the predictive models. The best performing model obtained a median precision of 100% at 10% recall and of 64% at 40% recall across all five bacterial species, and it outperformed previous published approaches on two benchmark datasets in terms of precision and recall. Our results indicate that even though there is limited sRNA sequence conservation across different bacterial species, there are intrinsic features in the genomic context of sRNAs that are conserved across taxa. We show that these features are utilized by machine learning approaches to learn a species-independent model to prioritize bona fide bacterial sRNAs.

## 19. Multispectral imaging for presymptomatic analysis of light leaf spot in oilseed rape.
**Keywords:** **'Disease detection', 'Light leaf spot', 'Machine learning', 'Multispectral', 'Novelty detection', 'Oilseed rape', 'Orientation effects', 'Photometric stereo', 'Preprocessing', 'Support vector machine'**

*Journal: Plant methods* *Publication Date: 2019*

[https://plantmethods.biomedcentral.com/articles/10.1186/s13007-019-0389-9](https://plantmethods.biomedcentral.com/articles/10.1186/s13007-019-0389-9){: .btn}

*abstract:* The use of spectral imaging within the plant phenotyping and breeding community has been increasing due its utility as a non-invasive diagnostic tool. However, there is a lack of imaging systems targeted specifically at plant science duties, resulting in low precision for canopy-scale measurements. This study trials a prototype multispectral system designed specifically for plant studies and looks at its use as an early detection system for visually asymptomatic disease phases, in this case 

## 20. Ontogeny of vocal rhythms in harbor seal pups: an exploratory study.
**Keywords:** **'bioacoustics', 'pinnipeds', 'rhythm', 'timing', 'vocal development'**

*Journal: Current zoology* *Publication Date: 2019-Feb*



*abstract:* Puppyhood is a very active social and vocal period in a harbor seal's life 

## 21. Rhythm and synchrony in animal movement and communication.


*Journal: Current zoology* *Publication Date: 2019-Feb*



*abstract:* 

## 22. FLOating-Window Projective Separator (FloWPS): A Data Trimming Tool for Support Vector Machines (SVM) to Improve Robustness of the Classifier.
**Keywords:** **'bioinformatics', 'gene expression', 'machine learning', 'oncology', 'personalized medicine', 'support vector machines'**

*Journal: Frontiers in genetics* *Publication Date: 2018*

[https://dx.doi.org/10.3389/fgene.2018.00717](https://dx.doi.org/10.3389/fgene.2018.00717){: .btn}

*abstract:* Here, we propose a heuristic technique of data trimming for SVM termed 

## 23. Guidance of Navigating Honeybees by Learned Elongated Ground Structures.
**Keywords:** **'compass alignment', 'ground structures', 'guiding landmarks', 'navigation', 'object recognition', 'sun compass'**

*Journal: Frontiers in behavioral neuroscience* *Publication Date: 2018*

[https://dx.doi.org/10.3389/fnbeh.2018.00322](https://dx.doi.org/10.3389/fnbeh.2018.00322){: .btn}

*abstract:* Elongated landscape features like forest edges, rivers, roads or boundaries of fields are particularly salient landmarks for navigating animals. Here, we ask how honeybees learn such structures and how they are used during their homing flights after being released at an unexpected location (catch-and-release paradigm). The experiments were performed in two landscapes that differed with respect to their overall structure: a rather feature-less landscape, and one rich in close and far distant landmarks. We tested three different forms of learning: learning during orientation flights, learning during training to a feeding site, and learning during homing flights after release at an unexpected site within the explored area. We found that bees use elongated ground structures, e.g., a field boundary separating two pastures close to the hive (Experiment 1), an irrigation channel (Experiment 2), a hedgerow along which the bees were trained (Experiment 3), a gravel road close to the hive and the feeder (Experiment 4), a path along an irrigation channel with its vegetation close to the feeder (Experiment 5) and a gravel road along which bees performed their homing flights (Experiment 6). Discrimination and generalization between the learned linear landmarks and similar ones in the test area depend on their object properties (irrigation channel, gravel road, hedgerow) and their compass orientation. We conclude that elongated ground structures are embedded into multiple landscape features indicating that memory of these linear structures is one component of bee navigation. Elongated structures interact and compete with other references. Object identification is an important part of this process. The objects are characterized not only by their appearance but also by their alignment in the compass. Their salience is highest if both components are close to what had been learned. High similarity in appearance can compensate for (partial) compass misalignment, and vice versa.

## 24. A Fast and Refined Cancer Regions Segmentation Framework in Whole-slide Breast Pathological Images.


*Journal: Scientific reports* *Publication Date: 2019-Jan-29*

[http://dx.doi.org/10.1038/s41598-018-37492-9](http://dx.doi.org/10.1038/s41598-018-37492-9){: .btn}

*abstract:* Supervised learning methods are commonly applied in medical image analysis. However, the success of these approaches is highly dependent on the availability of large manually detailed annotated dataset. Thus an automatic refined segmentation of whole-slide image (WSI) is significant to alleviate the annotation workload of pathologists. But most of the current ways can only output a rough prediction of lesion areas and consume much time in each slide. In this paper, we propose a fast and refined cancer regions segmentation framework v3_DCNN, which first preselects tumor regions using a classification model Inception-v3 and then employs a semantic segmentation model DCNN for refined segmentation. Our framework can generate a dense likelihood heatmap with the 1/8 side of original WSI in 11.5 minutes on the Camelyon16 dataset, which saves more than one hour for each WSI compared with the initial DCNN model. Experimental results show that our approach achieves a higher FROC score 83.5% with the champion's method of Camelyon16 challenge 80.7%. Based on v3 DCNN model, we further automatically produce heatmap of WSI and extract polygons of lesion regions for doctors, which is very helpful for their pathological diagnosis, detailed annotation and thus contributes to developing a more powerful deep learning model.

## 25. Towards reconstructing intelligible speech from the human auditory cortex.


*Journal: Scientific reports* *Publication Date: 2019-Jan-29*

[http://dx.doi.org/10.1038/s41598-018-37359-z](http://dx.doi.org/10.1038/s41598-018-37359-z){: .btn}

*abstract:* Auditory stimulus reconstruction is a technique that finds the best approximation of the acoustic stimulus from the population of evoked neural activity. Reconstructing speech from the human auditory cortex creates the possibility of a speech neuroprosthetic to establish a direct communication with the brain and has been shown to be possible in both overt and covert conditions. However, the low quality of the reconstructed speech has severely limited the utility of this method for brain-computer interface (BCI) applications. To advance the state-of-the-art in speech neuroprosthesis, we combined the recent advances in deep learning with the latest innovations in speech synthesis technologies to reconstruct closed-set intelligible speech from the human auditory cortex. We investigated the dependence of reconstruction accuracy on linear and nonlinear (deep neural network) regression methods and the acoustic representation that is used as the target of reconstruction, including auditory spectrogram and speech synthesis parameters. In addition, we compared the reconstruction accuracy from low and high neural frequency ranges. Our results show that a deep neural network model that directly estimates the parameters of a speech synthesizer from all neural frequencies achieves the highest subjective and objective scores on a digit recognition task, improving the intelligibility by 65% over the baseline method which used linear regression to reconstruct the auditory spectrogram. These results demonstrate the efficacy of deep learning and speech synthesis algorithms for designing the next generation of speech BCI systems, which not only can restore communications for paralyzed patients but also have the potential to transform human-computer interaction technologies.

## 26. Epithelium segmentation using deep learning in H&E-stained prostate specimens with immunohistochemistry as reference standard.


*Journal: Scientific reports* *Publication Date: 2019-Jan-29*

[http://dx.doi.org/10.1038/s41598-018-37257-4](http://dx.doi.org/10.1038/s41598-018-37257-4){: .btn}

*abstract:* Given the importance of gland morphology in grading prostate cancer (PCa), automatically differentiating between epithelium and other tissues is an important prerequisite for the development of automated methods for detecting PCa. We propose a new deep learning method to segment epithelial tissue in digitised hematoxylin and eosin (H&E) stained prostatectomy slides using immunohistochemistry (IHC) as reference standard. We used IHC to create a precise and objective ground truth compared to manual outlining on H&E slides, especially in areas with high-grade PCa. 102 tissue sections were stained with H&E and subsequently restained with P63 and CK8/18 IHC markers to highlight epithelial structures. Afterwards each pair was co-registered. First, we trained a U-Net to segment epithelial structures in IHC using a subset of the IHC slides that were preprocessed with color deconvolution. Second, this network was applied to the remaining slides to create the reference standard used to train a second U-Net on H&E. Our system accurately segmented both intact glands and individual tumour epithelial cells. The generalisation capacity of our system is shown using an independent external dataset from a different centre. We envision this segmentation as the first part of a fully automated prostate cancer grading pipeline.

## 27. Sequence Characteristics Distinguish Transcribed Enhancers from Promoters and Predict Their Breadth of Activity.
**Keywords:** **'enhancers', 'gene regulation', 'machine learning', 'promoters', 'sequence analysis'**

*Journal: Genetics* *Publication Date: 2019-Jan-29*

[http://www.genetics.org/cgi/pmidlookup?view=long&pmid=30696717](http://www.genetics.org/cgi/pmidlookup?view=long&pmid=30696717){: .btn}

*abstract:* Enhancers and promoters both regulate gene expression by recruiting transcription factors; however, the degree to which enhancer vs. promoter activity is due to differences in their sequences or to genomic context is the subject of ongoing debate. We examined this question by analyzing the sequences of thousands of transcribed enhancers and promoters from hundreds of cellular contexts previously identified by Cap Analysis of Gene Expression (CAGE). Support vector machine (SVM) classifiers trained on counts of all possible 6-bp-long sequences (6-mers) were able to accurately distinguish promoters from enhancers and distinguish their breadth of activity across tissues. Classifiers trained to predict enhancer activity also performed well when applied to promoter prediction tasks, but promoter-trained classifiers performed poorly on enhancers. This suggests that the learned sequence patterns predictive of enhancer activity generalize to promoters, but not vice versa. Our classifiers also indicate that there are functionally relevant differences in enhancer and promoter GC content beyond the influence of CpG islands. Furthermore, sequences characteristic of broad promoter or broad enhancer activity matched different transcription factors (TFs), with predicted ETS and RFX binding sites indicative of promoters and AP-1 sites indicative of enhancers. Finally, we evaluated the ability of our models to distinguish enhancers and promoters defined by histone modifications. Separating these classes was substantially more difficult, and this difference may contribute to ongoing debates about the similarity of enhancers and promoters. In summary, our results suggest that high confidence transcribed enhancers and promoters can largely be distinguished based on biologically relevant sequence properties.

## 28. Integrative analysis identifies potential DNA methylation biomarkers for pan-cancer diagnosis and prognosis.
**Keywords:** **'DNA methylation', 'cancer diagnosis', 'machine learning', 'pan-cancer', 'prognosis', 'survival analysis'**

*Journal: Epigenetics* *Publication Date: 2019-Jan-29*

[http://www.tandfonline.com/doi/full/10.1080/15592294.2019.1568178](http://www.tandfonline.com/doi/full/10.1080/15592294.2019.1568178){: .btn}

*abstract:* DNA methylation status is closely associated with diverse diseases, and is generally more stable than gene expression, thus abnormal DNA methylation could be important biomarkers for tumor diagnosis, treatment and prognosis. However, the signatures regarding DNA methylation changes for pan-cancer diagnosis and prognosis are less explored. Here we systematically analyzed the genome-wide DNA methylation patterns in diverse TCGA cancers with machine learning. We identified seven CpG sites that could effectively discriminate tumor samples from adjacent normal tissue samples for 12 main cancers of TCGA (1216 samples, AUC > 0.99). Those seven potential diagnostic biomarkers were further validated in the other 9 different TCGA cancers and 4 independent datasets (AUC > 0.92). Three out of the seven CpG sites were correlated with cell division, DNA replication and cell cycle. We also identified 12 CpG sites that can effectively distinguish 26 different cancers (7605 samples), and the result was repeatable in independent datasets as well as two disparate tumors with metastases (micro-average AUC > 0.89). Furthermore, a series of potential signatures that could significantly predict the prognosis of tumor patients for 7 different cancer were identified via survival analysis (p-value < 1e-4). Collectively, DNA methylation patterns vary greatly between tumor and adjacent normal tissues, as well as among different types of cancers. Our identified signatures may aid the decision of clinical diagnosis and prognosis for pan-cancer and the potential cancer-specific biomarkers could be used to predict the primary site of metastatic breast and prostate cancers.

## 29. Application of machine learning to predict obstructive sleep apnea syndrome severity.
**Keywords:** **'machine learning', 'obstructive sleep apnea syndrome'**

*Journal: Health informatics journal* *Publication Date: 2019-Jan-30*

[http://journals.sagepub.com/doi/full/10.1177/1460458218824725?url_ver=Z39.88-2003&rfr_id=ori:rid:crossref.org&rfr_dat=cr_pub%3dpubmed](http://journals.sagepub.com/doi/full/10.1177/1460458218824725?url_ver=Z39.88-2003&rfr_id=ori:rid:crossref.org&rfr_dat=cr_pub%3dpubmed){: .btn}

*abstract:* Obstructive sleep apnea syndrome has become an important public health concern. Polysomnography is traditionally considered an established and effective diagnostic tool providing information on the severity of obstructive sleep apnea syndrome and the degree of sleep fragmentation. However, the numerous steps in the polysomnography test to diagnose obstructive sleep apnea syndrome are costly and time consuming. This study aimed to test the efficacy and clinical applicability of different machine learning methods based on demographic information and questionnaire data to predict obstructive sleep apnea syndrome severity.

## 30. Could advances in representation learning in Artificial Intelligence provide the new paradigm for data integration in drug discovery?
**Keywords:** **'Artificial Intelligence', 'Autoencoders', 'Data integration', 'deep learning', 'drug discovery', 'neural networks', 'representation learning'**

*Journal: Expert opinion on drug discovery* *Publication Date: 2019-Jan-30*

[http://www.tandfonline.com/doi/full/10.1080/17460441.2019.1573811](http://www.tandfonline.com/doi/full/10.1080/17460441.2019.1573811){: .btn}

*abstract:* 

## 31. Illumination pattern design with deep learning for single-shot Fourier ptychographic microscopy.


*Journal: Optics express* *Publication Date: 2019-Jan-21*



*abstract:* Fourier ptychographic microscopy allows for the collection of images with a high space-bandwidth product at the cost of temporal resolution. In Fourier ptychographic microscopy, the light source of a conventional widefield microscope is replaced with a light-emitting diode (LED) matrix, and multiple images are collected with different LED illumination patterns. From these images, a higher-resolution image can be computationally reconstructed without sacrificing field-of-view. We use deep learning to achieve single-shot imaging without sacrificing the space-bandwidth product, reducing the acquisition time in Fourier ptychographic microscopy by a factor of 69. In our deep learning approach, a training dataset of high-resolution images is used to jointly optimize a single LED illumination pattern with the parameters of a reconstruction algorithm. Our work paves the way for high-throughput imaging in biological studies.

## 32. Large-Scale Assessment of Bioinformatics Tools for Lysine Succinylation Sites.
**Keywords:** **'feature descriptor', 'lysine succinylation', 'machine learning', 'sequence analysis', 'tool development'**

*Journal: Cells* *Publication Date: 2019-Jan-28*

[http://www.mdpi.com/resolver?pii=cells8020095](http://www.mdpi.com/resolver?pii=cells8020095){: .btn}

*abstract:* Lysine succinylation is a form of posttranslational modification of the proteins that play an essential functional role in every aspect of cell metabolism in both prokaryotes and eukaryotes. Aside from experimental identification of succinylation sites, there has been an intense effort geared towards the development of sequence-based prediction through machine learning, due to its promising and essential properties of being highly accurate, robust and cost-effective. In spite of these advantages, there are several problems that are in need of attention in the design and development of succinylation site predictors. Notwithstanding of many studies on the employment of machine learning approaches, few articles have examined this bioinformatics field in a systematic manner. Thus, we review the advancements regarding the current state-of-the-art prediction models, datasets, and online resources and illustrate the challenges and limitations to present a useful guideline for developing powerful succinylation site prediction tools.

## 33. Optimizing the Predictive Ability of Machine Learning Methods for Landslide Susceptibility Mapping Using SMOTE for Lishui City in Zhejiang Province, China.
**Keywords:** **'Lishui City', 'SMOTE', 'landslide susceptibility', 'machine learning', 'neighborhood rough set theory', 'slope units'**

*Journal: International journal of environmental research and public health* *Publication Date: 2019-Jan-28*

[http://www.mdpi.com/resolver?pii=ijerph16030368](http://www.mdpi.com/resolver?pii=ijerph16030368){: .btn}

*abstract:* The main goal of this study was to use the synthetic minority oversampling technique (SMOTE) to expand the quantity of landslide samples for machine learning methods (i.e., support vector machine (SVM), logistic regression (LR), artiﬁcial neural network (ANN), and random forest (RF)) to produce high-quality landslide susceptibility maps for Lishui City in Zhejiang Province, China. Landslide-related factors were extracted from topographic maps, geological maps, and satellite images. Twelve factors were selected as independent variables using correlation coefficient analysis and the neighborhood rough set (NRS) method. In total, 288 soil landslides were mapped using field surveys, historical records, and satellite images. The landslides were randomly divided into two datasets: 70% of all landslides were selected as the original training dataset and 30% were used for validation. Then, SMOTE was employed to generate datasets with sizes ranging from two to thirty times that of the training dataset to establish and compare the four machine learning methods for landslide susceptibility mapping. In addition, we used slope units to subdivide the terrain to determine the landslide susceptibility. Finally, the landslide susceptibility maps were validated using statistical indexes and the area under the curve (AUC). The results indicated that the performances of the four machine learning methods showed different levels of improvement as the sample sizes increased. The RF model exhibited a more substantial improvement (AUC improved by 24.12%) than did the ANN (18.94%), SVM (17.77%), and LR (3.00%) models. Furthermore, the ANN model achieved the highest predictive ability (AUC = 0.98), followed by the RF (AUC = 0.96), SVM (AUC = 0.94), and LR (AUC = 0.79) models. This approach significantly improves the performance of machine learning techniques for landslide susceptibility mapping, thereby providing a better tool for reducing the impacts of landslide disasters.

## 34. A Hierarchical Deep Fusion Framework for Egocentric Activity Recognition Using a Wearable Hybrid Sensor System.
**Keywords:** **'deep learning', 'egocentric activity recognition', 'hierarchical fusion framework', 'wearable sensor system'**

*Journal: Sensors (Basel, Switzerland)* *Publication Date: 2019-Jan-28*

[http://www.mdpi.com/resolver?pii=s19030546](http://www.mdpi.com/resolver?pii=s19030546){: .btn}

*abstract:* Recently, egocentric activity recognition has attracted considerable attention in the pattern recognition and artificial intelligence communities because of its wide applicability in medical care, smart homes, and security monitoring. In this study, we developed and implemented a deep-learning-based hierarchical fusion framework for the recognition of egocentric activities of daily living (ADLs) in a wearable hybrid sensor system comprising motion sensors and cameras. Long short-term memory (LSTM) and a convolutional neural network are used to perform egocentric ADL recognition based on motion sensor data and photo streaming in different layers, respectively. The motion sensor data are used solely for activity classification according to motion state, while the photo stream is used for further specific activity recognition in the motion state groups. Thus, both motion sensor data and photo stream work in their most suitable classification mode to significantly reduce the negative influence of sensor differences on the fusion results. Experimental results show that the proposed method not only is more accurate than the existing direct fusion method (by up to 6%) but also avoids the time-consuming computation of optical flow in the existing method, which makes the proposed algorithm less complex and more suitable for practical application.

## 35. Machine Learning and Integrative Analysis of Biomedical Big Data.
**Keywords:** **'class imbalance', 'curse of dimensionality', 'data integration', 'heterogeneous data', 'machine learning', 'missing data', 'multi-omics', 'scalability'**

*Journal: Genes* *Publication Date: 2019-Jan-28*

[http://www.mdpi.com/resolver?pii=genes10020087](http://www.mdpi.com/resolver?pii=genes10020087){: .btn}

*abstract:* Recent developments in high-throughput technologies have accelerated the accumulation of massive amounts of omics data from multiple sources: genome, epigenome, transcriptome, proteome, metabolome, etc. Traditionally, data from each source (e.g., genome) is analyzed in isolation using statistical and machine learning (ML) methods. Integrative analysis of multi-omics and clinical data is key to new biomedical discoveries and advancements in precision medicine. However, data integration poses new computational challenges as well as exacerbates the ones associated with single-omics studies. Specialized computational approaches are required to effectively and efficiently perform integrative analysis of biomedical data acquired from diverse modalities. In this review, we discuss state-of-the-art ML-based approaches for tackling five specific computational challenges associated with integrative analysis: curse of dimensionality, data heterogeneity, missing data, class imbalance and scalability issues.

## 36. An Artificial Intelligence Application for Post-Earthquake Damage Mapping in Palu, Central Sulawesi, Indonesia.
**Keywords:** **'ANN', 'Palu earthquake', 'SVM', 'post-earthquake damage map'**

*Journal: Sensors (Basel, Switzerland)* *Publication Date: 2019-Jan-28*

[http://www.mdpi.com/resolver?pii=s19030542](http://www.mdpi.com/resolver?pii=s19030542){: .btn}

*abstract:* A Mw 7.4 earthquake hit Donggala County, Central Sulawesi Province, Indonesia, on 28 September 2018, triggering a tsunami and liquefaction in Palu City and Donggala. Around 2101 fatalities ensued and 68,451 houses were damaged by the earthquake. In light of this devastating event, a post-earthquake map is required to establish the first step in the evacuation and mitigation plan. In this study, remote sensing imagery from the Landsat-8 and Sentinel-2 satellites was used. Pre- and post-earthquake satellite images were classified using artificial neural network (ANN) and support vector machine (SVM) classifiers and processed using a decorrelation method to generate the post-earthquake damage map. The affected areas were compared to the field data, the percentage conformity between the ANN and SVM results was analyzed, and four post-earthquake damage maps were generated. Based on the conformity analysis, the Landsat-8 imagery (85.83%) was superior to that of Sentinel-2 (63.88%). The resulting post-earthquake damage map can be used to assess the distribution of seismic damage following the Palu earthquake and may be used to mitigate damage in the event of future earthquakes.

